<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.1//EN" "https://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd">
<html xmlns="https://www.w3.org/1999/xhtml">
<head>
<meta http-equiv="content-type" content="text/html; charset=UTF-8"/>
<title>TestNG Report</title>
<style type="text/css">table {margin-bottom:10px;border-collapse:collapse;empty-cells:show}th,td {border:1px solid #009;padding:.25em .5em}th {vertical-align:bottom}td {vertical-align:top}table a {font-weight:bold}.stripe td {background-color: #E6EBF9}.num {text-align:right}.passedodd td {background-color: #3F3}.passedeven td {background-color: #0A0}.skippedodd td {background-color: #DDD}.skippedeven td {background-color: #CCC}.failedodd td,.attn {background-color: #F33}.failedeven td,.stripe .attn {background-color: #D00}.stacktrace {white-space:pre;font-family:monospace}.totop {font-size:85%;text-align:center;border-bottom:2px solid #000}.invisible {display:none}</style>
</head>
<body>
<table>
<tr><th>Test</th><th># Passed</th><th># Skipped</th><th># Retried</th><th># Failed</th><th>Time (ms)</th><th>Included Groups</th><th>Excluded Groups</th></tr>
<tr><th colspan="8">Briskpe-Cross-Platform-Suite</th></tr>
<tr><td><a href="#t0">iOS</a></td><td class="num">0</td><td class="num attn">4</td><td class="num">0</td><td class="num">0</td><td class="num">1,976</td><td></td><td></td></tr>
<tr class="stripe"><td><a href="#t1">Android</a></td><td class="num">0</td><td class="num attn">4</td><td class="num">0</td><td class="num">0</td><td class="num">1,965</td><td></td><td></td></tr>
<tr><td><a href="#t2">Web</a></td><td class="num">0</td><td class="num attn">4</td><td class="num">0</td><td class="num">0</td><td class="num">2,093</td><td></td><td></td></tr>
<tr><th>Total</th><th class="num">0</th><th class="num attn">12</th><th class="num">0</th><th class="num">0</th><th class="num">6,034</th><th colspan="2"></th></tr>
</table>
<table id='summary'><thead><tr><th>Class</th><th>Method</th><th>Start</th><th>Time (ms)</th></tr></thead><tbody><tr><th colspan="4">Briskpe-Cross-Platform-Suite</th></tr></tbody><tbody id="t0"><tr><th colspan="4">iOS &#8212; failed (configuration methods)</th></tr><tr class="failedeven"><td rowspan="6">com.briskpe.tests.DashBoardTest</td><td><a href="#m0">setUp</a></td><td rowspan="2">1753104523543</td><td rowspan="2">79</td></tr><tr class="failedeven"><td><a href="#m1">setUp</a></td></tr><tr class="failedeven"><td><a href="#m2">tearDown</a></td><td rowspan="2">1753104523781</td><td rowspan="2">4</td></tr><tr class="failedeven"><td><a href="#m3">tearDown</a></td></tr><tr class="failedeven"><td><a href="#m4">tearDownTestWithScreenshot</a></td><td rowspan="2">1753104524114</td><td rowspan="2">0</td></tr><tr class="failedeven"><td><a href="#m5">tearDownTestWithScreenshot</a></td></tr><tr class="failedodd"><td rowspan="6">com.briskpe.tests.LoginTest</td><td><a href="#m6">setUp</a></td><td rowspan="2">1753104523198</td><td rowspan="2">152</td></tr><tr class="failedodd"><td><a href="#m7">setUp</a></td></tr><tr class="failedodd"><td><a href="#m8">tearDown</a></td><td rowspan="2">1753104523511</td><td rowspan="2">0</td></tr><tr class="failedodd"><td><a href="#m9">tearDown</a></td></tr><tr class="failedodd"><td><a href="#m10">tearDownLoginTest</a></td><td rowspan="2">1753104523116</td><td rowspan="2">1</td></tr><tr class="failedodd"><td><a href="#m11">tearDownLoginTest</a></td></tr><tr><th colspan="4">iOS &#8212; skipped (configuration methods)</th></tr><tr class="skippedeven"><td rowspan="1">com.briskpe.tests.DashBoardTest</td><td><a href="#m12">flushReport</a></td><td rowspan="1">1753104524582</td><td rowspan="1">0</td></tr><tr class="skippedodd"><td rowspan="1">com.briskpe.tests.LoginTest</td><td><a href="#m13">flushLoginReport</a></td><td rowspan="1">1753104524582</td><td rowspan="1">0</td></tr><tr><th colspan="4">iOS &#8212; skipped</th></tr><tr class="skippedeven"><td rowspan="2">com.briskpe.tests.DashBoardTest</td><td><a href="#m14">clickSkipButtonTest</a></td><td rowspan="1">1753104523670</td><td rowspan="1">0</td></tr><tr class="skippedeven"><td><a href="#m15">verifyUserCanNavigateToSideMenuItems</a></td><td rowspan="1">1753104524098</td><td rowspan="1">0</td></tr><tr class="skippedodd"><td rowspan="2">com.briskpe.tests.LoginTest</td><td><a href="#m16">shouldLoginWithValidMobile</a></td><td rowspan="1">1753104523092</td><td rowspan="1">1</td></tr><tr class="skippedodd"><td><a href="#m17">shouldNotLoginWithInvalidMobileNumber</a></td><td rowspan="1">1753104523369</td><td rowspan="1">0</td></tr></tbody>
<tbody id="t1"><tr><th colspan="4">Android &#8212; failed (configuration methods)</th></tr><tr class="failedeven"><td rowspan="6">com.briskpe.tests.DashBoardTest</td><td><a href="#m18">setUp</a></td><td rowspan="2">1753104523832</td><td rowspan="2">154</td></tr><tr class="failedeven"><td><a href="#m19">setUp</a></td></tr><tr class="failedeven"><td><a href="#m20">tearDown</a></td><td rowspan="2">1753104524142</td><td rowspan="2">0</td></tr><tr class="failedeven"><td><a href="#m21">tearDown</a></td></tr><tr class="failedeven"><td><a href="#m22">tearDownTestWithScreenshot</a></td><td rowspan="2">1753104523702</td><td rowspan="2">0</td></tr><tr class="failedeven"><td><a href="#m23">tearDownTestWithScreenshot</a></td></tr><tr class="failedodd"><td rowspan="6">com.briskpe.tests.LoginTest</td><td><a href="#m24">setUp</a></td><td rowspan="2">1753104522345</td><td rowspan="2">598</td></tr><tr class="failedodd"><td><a href="#m25">setUp</a></td></tr><tr class="failedodd"><td><a href="#m26">tearDown</a></td><td rowspan="2">1753104523170</td><td rowspan="2">0</td></tr><tr class="failedodd"><td><a href="#m27">tearDown</a></td></tr><tr class="failedodd"><td><a href="#m28">tearDownLoginTest</a></td><td rowspan="2">1753104523416</td><td rowspan="2">0</td></tr><tr class="failedodd"><td><a href="#m29">tearDownLoginTest</a></td></tr><tr><th colspan="4">Android &#8212; skipped</th></tr><tr class="skippedeven"><td rowspan="2">com.briskpe.tests.DashBoardTest</td><td><a href="#m30">clickSkipButtonTest</a></td><td rowspan="1">1753104523695</td><td rowspan="1">0</td></tr><tr class="skippedeven"><td><a href="#m31">verifyUserCanNavigateToSideMenuItems</a></td><td rowspan="1">1753104524003</td><td rowspan="1">0</td></tr><tr class="skippedodd"><td rowspan="2">com.briskpe.tests.LoginTest</td><td><a href="#m32">shouldLoginWithValidMobile</a></td><td rowspan="1">1753104523092</td><td rowspan="1">1</td></tr><tr class="skippedodd"><td><a href="#m33">shouldNotLoginWithInvalidMobileNumber</a></td><td rowspan="1">1753104523411</td><td rowspan="1">0</td></tr></tbody>
<tbody id="t2"><tr><th colspan="4">Web &#8212; failed (configuration methods)</th></tr><tr class="failedeven"><td rowspan="6">com.briskpe.tests.DashBoardTest</td><td><a href="#m34">setUp</a></td><td rowspan="2">1753104523558</td><td rowspan="2">112</td></tr><tr class="failedeven"><td><a href="#m35">setUp</a></td></tr><tr class="failedeven"><td><a href="#m36">tearDown</a></td><td rowspan="2">1753104523893</td><td rowspan="2">0</td></tr><tr class="failedeven"><td><a href="#m37">tearDown</a></td></tr><tr class="failedeven"><td><a href="#m38">tearDownTestWithScreenshot</a></td><td rowspan="2">1753104523814</td><td rowspan="2">0</td></tr><tr class="failedeven"><td><a href="#m39">tearDownTestWithScreenshot</a></td></tr><tr class="failedodd"><td rowspan="6">com.briskpe.tests.LoginTest</td><td><a href="#m40">setUp</a></td><td rowspan="2">1753104522345</td><td rowspan="2">598</td></tr><tr class="failedodd"><td><a href="#m41">setUp</a></td></tr><tr class="failedodd"><td><a href="#m42">tearDown</a></td><td rowspan="2">1753104523176</td><td rowspan="2">0</td></tr><tr class="failedodd"><td><a href="#m43">tearDown</a></td></tr><tr class="failedodd"><td><a href="#m44">tearDownLoginTest</a></td><td rowspan="2">1753104523416</td><td rowspan="2">0</td></tr><tr class="failedodd"><td><a href="#m45">tearDownLoginTest</a></td></tr><tr><th colspan="4">Web &#8212; skipped</th></tr><tr class="skippedeven"><td rowspan="2">com.briskpe.tests.DashBoardTest</td><td><a href="#m46">clickSkipButtonTest</a></td><td rowspan="1">1753104523781</td><td rowspan="1">0</td></tr><tr class="skippedeven"><td><a href="#m47">verifyUserCanNavigateToSideMenuItems</a></td><td rowspan="1">1753104524202</td><td rowspan="1">0</td></tr><tr class="skippedodd"><td rowspan="2">com.briskpe.tests.LoginTest</td><td><a href="#m48">shouldLoginWithValidMobile</a></td><td rowspan="1">1753104523091</td><td rowspan="1">1</td></tr><tr class="skippedodd"><td><a href="#m49">shouldNotLoginWithInvalidMobileNumber</a></td><td rowspan="1">1753104523406</td><td rowspan="1">0</td></tr></tbody>
</table>
<h2>iOS</h2><h3 id="m0">com.briskpe.tests.DashBoardTest#setUp</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th></tr><tr class="param stripe"><td>public void com.briskpe.tests.DashBoardTest.clickSkipButtonTest()</td><td>IOS</td></tr><tr><th colspan="2">Exception</th></tr><tr><td colspan="2"><div class="stacktrace">org.openqa.selenium.SessionNotCreatedException: Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure. 
Host info: host: &apos;VISHALSHUKLA&apos;, ip: &apos;10.188.60.117&apos;
Build info: version: &apos;4.34.0&apos;, revision: &apos;707dcb4246*&apos;
System info: os.name: &apos;Windows 11&apos;, os.arch: &apos;amd64&apos;, os.version: &apos;10.0&apos;, java.version: &apos;17.0.12&apos;
Driver info: io.appium.java_client.ios.IOSDriver
Command: [null, newSession {capabilities=[Capabilities {appium:automationName: Flutter, appium:bundleId: com.briskpe.app, platformName: IOS}]}]
Capabilities {appium:automationName: Flutter, appium:bundleId: com.briskpe.app, platformName: IOS}
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:562)
	at io.appium.java_client.AppiumDriver.startSession(AppiumDriver.java:313)
	at org.openqa.selenium.remote.RemoteWebDriver.&lt;init&gt;(RemoteWebDriver.java:173)
	at io.appium.java_client.AppiumDriver.&lt;init&gt;(AppiumDriver.java:101)
	at io.appium.java_client.AppiumDriver.&lt;init&gt;(AppiumDriver.java:113)
	at io.appium.java_client.ios.IOSDriver.&lt;init&gt;(IOSDriver.java:101)
	at com.briskpe.framework.core.DriverFactory.createIOSDriver(DriverFactory.java:120)
	at com.briskpe.framework.core.DriverFactory.createDriver(DriverFactory.java:52)
	at com.briskpe.framework.core.DriverFactory.createDriver(DriverFactory.java:40)
	at com.briskpe.tests.BaseTest.setUp(BaseTest.java:39)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:842)
Caused by: java.io.UncheckedIOException: java.net.ConnectException
	at org.openqa.selenium.remote.http.jdk.JdkHttpClient.execute0(JdkHttpClient.java:500)
	at io.appium.java_client.internal.filters.AppiumIdempotencyFilter.lambda$apply$0(AppiumIdempotencyFilter.java:35)
	at io.appium.java_client.internal.filters.AppiumUserAgentFilter.lambda$apply$0(AppiumUserAgentFilter.java:86)
	at org.openqa.selenium.remote.http.Filter.lambda$andFinally$1(Filter.java:55)
	at org.openqa.selenium.remote.http.jdk.JdkHttpClient.lambda$executeAsync$2(JdkHttpClient.java:387)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
	... 4 more
Caused by: java.net.ConnectException
	at java.net.http/jdk.internal.net.http.HttpClientImpl.send(HttpClientImpl.java:574)
	at java.net.http/jdk.internal.net.http.HttpClientFacade.send(HttpClientFacade.java:123)
	at org.openqa.selenium.remote.http.jdk.JdkHttpClient.execute0(JdkHttpClient.java:459)
	... 9 more
Caused by: java.net.ConnectException
	at java.net.http/jdk.internal.net.http.common.Utils.toConnectException(Utils.java:1055)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.connectAsync(PlainHttpConnection.java:198)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.checkRetryConnect(PlainHttpConnection.java:230)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.lambda$connectAsync$1(PlainHttpConnection.java:206)
	at java.base/java.util.concurrent.CompletableFuture.uniHandle(CompletableFuture.java:934)
	at java.base/java.util.concurrent.CompletableFuture$UniHandle.tryFire(CompletableFuture.java:911)
	at java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1773)
	... 3 more
Caused by: java.nio.channels.ClosedChannelException
	at java.base/sun.nio.ch.SocketChannelImpl.ensureOpen(SocketChannelImpl.java:195)
	at java.base/sun.nio.ch.SocketChannelImpl.beginConnect(SocketChannelImpl.java:760)
	at java.base/sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:848)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.lambda$connectAsync$0(PlainHttpConnection.java:183)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:569)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.connectAsync(PlainHttpConnection.java:185)
	... 9 more
... Removed 21 stack frames</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p>
<h3 id="m1">com.briskpe.tests.DashBoardTest#setUp</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th></tr><tr class="param stripe"><td>public void com.briskpe.tests.DashBoardTest.verifyUserCanNavigateToSideMenuItems()</td><td>IOS</td></tr><tr><th colspan="2">Exception</th></tr><tr><td colspan="2"><div class="stacktrace">org.openqa.selenium.SessionNotCreatedException: Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure. 
Host info: host: &apos;VISHALSHUKLA&apos;, ip: &apos;10.188.60.117&apos;
Build info: version: &apos;4.34.0&apos;, revision: &apos;707dcb4246*&apos;
System info: os.name: &apos;Windows 11&apos;, os.arch: &apos;amd64&apos;, os.version: &apos;10.0&apos;, java.version: &apos;17.0.12&apos;
Driver info: io.appium.java_client.ios.IOSDriver
Command: [null, newSession {capabilities=[Capabilities {appium:automationName: Flutter, appium:bundleId: com.briskpe.app, platformName: IOS}]}]
Capabilities {appium:automationName: Flutter, appium:bundleId: com.briskpe.app, platformName: IOS}
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:562)
	at io.appium.java_client.AppiumDriver.startSession(AppiumDriver.java:313)
	at org.openqa.selenium.remote.RemoteWebDriver.&lt;init&gt;(RemoteWebDriver.java:173)
	at io.appium.java_client.AppiumDriver.&lt;init&gt;(AppiumDriver.java:101)
	at io.appium.java_client.AppiumDriver.&lt;init&gt;(AppiumDriver.java:113)
	at io.appium.java_client.ios.IOSDriver.&lt;init&gt;(IOSDriver.java:101)
	at com.briskpe.framework.core.DriverFactory.createIOSDriver(DriverFactory.java:120)
	at com.briskpe.framework.core.DriverFactory.createDriver(DriverFactory.java:52)
	at com.briskpe.framework.core.DriverFactory.createDriver(DriverFactory.java:40)
	at com.briskpe.tests.BaseTest.setUp(BaseTest.java:39)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:842)
Caused by: java.io.UncheckedIOException: java.net.ConnectException
	at org.openqa.selenium.remote.http.jdk.JdkHttpClient.execute0(JdkHttpClient.java:500)
	at io.appium.java_client.internal.filters.AppiumIdempotencyFilter.lambda$apply$0(AppiumIdempotencyFilter.java:35)
	at io.appium.java_client.internal.filters.AppiumUserAgentFilter.lambda$apply$0(AppiumUserAgentFilter.java:86)
	at org.openqa.selenium.remote.http.Filter.lambda$andFinally$1(Filter.java:55)
	at org.openqa.selenium.remote.http.jdk.JdkHttpClient.lambda$executeAsync$2(JdkHttpClient.java:387)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
	... 4 more
Caused by: java.net.ConnectException
	at java.net.http/jdk.internal.net.http.HttpClientImpl.send(HttpClientImpl.java:574)
	at java.net.http/jdk.internal.net.http.HttpClientFacade.send(HttpClientFacade.java:123)
	at org.openqa.selenium.remote.http.jdk.JdkHttpClient.execute0(JdkHttpClient.java:459)
	... 9 more
Caused by: java.net.ConnectException
	at java.net.http/jdk.internal.net.http.common.Utils.toConnectException(Utils.java:1055)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.connectAsync(PlainHttpConnection.java:198)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.checkRetryConnect(PlainHttpConnection.java:230)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.lambda$connectAsync$1(PlainHttpConnection.java:206)
	at java.base/java.util.concurrent.CompletableFuture.uniHandle(CompletableFuture.java:934)
	at java.base/java.util.concurrent.CompletableFuture$UniHandle.tryFire(CompletableFuture.java:911)
	at java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1773)
	... 3 more
Caused by: java.nio.channels.ClosedChannelException
	at java.base/sun.nio.ch.SocketChannelImpl.ensureOpen(SocketChannelImpl.java:195)
	at java.base/sun.nio.ch.SocketChannelImpl.beginConnect(SocketChannelImpl.java:760)
	at java.base/sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:848)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.lambda$connectAsync$0(PlainHttpConnection.java:183)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:569)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.connectAsync(PlainHttpConnection.java:185)
	... 9 more
... Removed 21 stack frames</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p>
<h3 id="m2">com.briskpe.tests.DashBoardTest#tearDown</h3><table class="result"><tr class="param"><th>Parameter #1</th></tr><tr class="param stripe"><td>[TestResult name=clickSkipButtonTest status=SKIP method=DashBoardTest.clickSkipButtonTest()[pri:1, instance:com.briskpe.tests.DashBoardTest@f68f0dc] output={null}]</td></tr><tr><th>Exception</th></tr><tr><td><div class="stacktrace">java.lang.NullPointerException: Cannot invoke &quot;com.aventstack.extentreports.ExtentTest.skip(String)&quot; because &quot;com.briskpe.tests.BaseTest.test&quot; is null
	at com.briskpe.tests.BaseTest.tearDown(BaseTest.java:64)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:842)
... Removed 22 stack frames</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p>
<h3 id="m3">com.briskpe.tests.DashBoardTest#tearDown</h3><table class="result"><tr class="param"><th>Parameter #1</th></tr><tr class="param stripe"><td>[TestResult name=verifyUserCanNavigateToSideMenuItems status=SKIP method=DashBoardTest.verifyUserCanNavigateToSideMenuItems()[pri:2, instance:com.briskpe.tests.DashBoardTest@f68f0dc] output={null}]</td></tr><tr><th>Exception</th></tr><tr><td><div class="stacktrace">java.lang.NullPointerException: Cannot invoke &quot;com.aventstack.extentreports.ExtentTest.skip(String)&quot; because &quot;com.briskpe.tests.BaseTest.test&quot; is null
	at com.briskpe.tests.BaseTest.tearDown(BaseTest.java:64)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:842)
... Removed 22 stack frames</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p>
<h3 id="m4">com.briskpe.tests.DashBoardTest#tearDownTestWithScreenshot</h3><table class="result"><tr class="param"><th>Parameter #1</th></tr><tr class="param stripe"><td>[TestResult name=verifyUserCanNavigateToSideMenuItems status=SKIP method=DashBoardTest.verifyUserCanNavigateToSideMenuItems()[pri:2, instance:com.briskpe.tests.DashBoardTest@f68f0dc] output={null}]</td></tr><tr><th>Exception</th></tr><tr><td><div class="stacktrace">java.lang.NullPointerException: Cannot invoke &quot;com.aventstack.extentreports.ExtentTest.skip(String)&quot; because &quot;com.briskpe.tests.DashBoardTest.test&quot; is null
	at com.briskpe.tests.DashBoardTest.tearDownTestWithScreenshot(DashBoardTest.java:89)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:842)
... Removed 22 stack frames</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p>
<h3 id="m5">com.briskpe.tests.DashBoardTest#tearDownTestWithScreenshot</h3><table class="result"><tr class="param"><th>Parameter #1</th></tr><tr class="param stripe"><td>[TestResult name=clickSkipButtonTest status=SKIP method=DashBoardTest.clickSkipButtonTest()[pri:1, instance:com.briskpe.tests.DashBoardTest@f68f0dc] output={null}]</td></tr><tr><th>Exception</th></tr><tr><td><div class="stacktrace">java.lang.NullPointerException: Cannot invoke &quot;com.aventstack.extentreports.ExtentTest.skip(String)&quot; because &quot;com.briskpe.tests.DashBoardTest.test&quot; is null
	at com.briskpe.tests.DashBoardTest.tearDownTestWithScreenshot(DashBoardTest.java:89)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:842)
... Removed 22 stack frames</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p>
<h3 id="m6">com.briskpe.tests.LoginTest#setUp</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th></tr><tr class="param stripe"><td>public void com.briskpe.tests.LoginTest.shouldNotLoginWithInvalidMobileNumber()</td><td>IOS</td></tr><tr><th colspan="2">Exception</th></tr><tr><td colspan="2"><div class="stacktrace">org.openqa.selenium.SessionNotCreatedException: Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure. 
Host info: host: &apos;VISHALSHUKLA&apos;, ip: &apos;10.188.60.117&apos;
Build info: version: &apos;4.34.0&apos;, revision: &apos;707dcb4246*&apos;
System info: os.name: &apos;Windows 11&apos;, os.arch: &apos;amd64&apos;, os.version: &apos;10.0&apos;, java.version: &apos;17.0.12&apos;
Driver info: io.appium.java_client.ios.IOSDriver
Command: [null, newSession {capabilities=[Capabilities {appium:automationName: Flutter, appium:bundleId: com.briskpe.app, platformName: IOS}]}]
Capabilities {appium:automationName: Flutter, appium:bundleId: com.briskpe.app, platformName: IOS}
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:562)
	at io.appium.java_client.AppiumDriver.startSession(AppiumDriver.java:313)
	at org.openqa.selenium.remote.RemoteWebDriver.&lt;init&gt;(RemoteWebDriver.java:173)
	at io.appium.java_client.AppiumDriver.&lt;init&gt;(AppiumDriver.java:101)
	at io.appium.java_client.AppiumDriver.&lt;init&gt;(AppiumDriver.java:113)
	at io.appium.java_client.ios.IOSDriver.&lt;init&gt;(IOSDriver.java:101)
	at com.briskpe.framework.core.DriverFactory.createIOSDriver(DriverFactory.java:120)
	at com.briskpe.framework.core.DriverFactory.createDriver(DriverFactory.java:52)
	at com.briskpe.framework.core.DriverFactory.createDriver(DriverFactory.java:40)
	at com.briskpe.tests.BaseTest.setUp(BaseTest.java:39)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:842)
Caused by: java.io.UncheckedIOException: java.net.ConnectException
	at org.openqa.selenium.remote.http.jdk.JdkHttpClient.execute0(JdkHttpClient.java:500)
	at io.appium.java_client.internal.filters.AppiumIdempotencyFilter.lambda$apply$0(AppiumIdempotencyFilter.java:35)
	at io.appium.java_client.internal.filters.AppiumUserAgentFilter.lambda$apply$0(AppiumUserAgentFilter.java:86)
	at org.openqa.selenium.remote.http.Filter.lambda$andFinally$1(Filter.java:55)
	at org.openqa.selenium.remote.http.jdk.JdkHttpClient.lambda$executeAsync$2(JdkHttpClient.java:387)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
	... 4 more
Caused by: java.net.ConnectException
	at java.net.http/jdk.internal.net.http.HttpClientImpl.send(HttpClientImpl.java:574)
	at java.net.http/jdk.internal.net.http.HttpClientFacade.send(HttpClientFacade.java:123)
	at org.openqa.selenium.remote.http.jdk.JdkHttpClient.execute0(JdkHttpClient.java:459)
	... 9 more
Caused by: java.net.ConnectException
	at java.net.http/jdk.internal.net.http.common.Utils.toConnectException(Utils.java:1055)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.connectAsync(PlainHttpConnection.java:198)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.checkRetryConnect(PlainHttpConnection.java:230)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.lambda$connectAsync$1(PlainHttpConnection.java:206)
	at java.base/java.util.concurrent.CompletableFuture.uniHandle(CompletableFuture.java:934)
	at java.base/java.util.concurrent.CompletableFuture$UniHandle.tryFire(CompletableFuture.java:911)
	at java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1773)
	... 3 more
Caused by: java.nio.channels.ClosedChannelException
	at java.base/sun.nio.ch.SocketChannelImpl.ensureOpen(SocketChannelImpl.java:195)
	at java.base/sun.nio.ch.SocketChannelImpl.beginConnect(SocketChannelImpl.java:760)
	at java.base/sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:848)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.lambda$connectAsync$0(PlainHttpConnection.java:183)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:569)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.connectAsync(PlainHttpConnection.java:185)
	... 9 more
... Removed 21 stack frames</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p>
<h3 id="m7">com.briskpe.tests.LoginTest#setUp</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th></tr><tr class="param stripe"><td>public void com.briskpe.tests.LoginTest.shouldLoginWithValidMobile()</td><td>IOS</td></tr><tr><th colspan="2">Exception</th></tr><tr><td colspan="2"><div class="stacktrace">org.openqa.selenium.SessionNotCreatedException: Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure. 
Host info: host: &apos;VISHALSHUKLA&apos;, ip: &apos;10.188.60.117&apos;
Build info: version: &apos;4.34.0&apos;, revision: &apos;707dcb4246*&apos;
System info: os.name: &apos;Windows 11&apos;, os.arch: &apos;amd64&apos;, os.version: &apos;10.0&apos;, java.version: &apos;17.0.12&apos;
Driver info: io.appium.java_client.ios.IOSDriver
Command: [null, newSession {capabilities=[Capabilities {appium:automationName: Flutter, appium:bundleId: com.briskpe.app, platformName: IOS}]}]
Capabilities {appium:automationName: Flutter, appium:bundleId: com.briskpe.app, platformName: IOS}
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:562)
	at io.appium.java_client.AppiumDriver.startSession(AppiumDriver.java:313)
	at org.openqa.selenium.remote.RemoteWebDriver.&lt;init&gt;(RemoteWebDriver.java:173)
	at io.appium.java_client.AppiumDriver.&lt;init&gt;(AppiumDriver.java:101)
	at io.appium.java_client.AppiumDriver.&lt;init&gt;(AppiumDriver.java:113)
	at io.appium.java_client.ios.IOSDriver.&lt;init&gt;(IOSDriver.java:101)
	at com.briskpe.framework.core.DriverFactory.createIOSDriver(DriverFactory.java:120)
	at com.briskpe.framework.core.DriverFactory.createDriver(DriverFactory.java:52)
	at com.briskpe.framework.core.DriverFactory.createDriver(DriverFactory.java:40)
	at com.briskpe.tests.BaseTest.setUp(BaseTest.java:39)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:842)
Caused by: java.io.UncheckedIOException: java.net.ConnectException
	at org.openqa.selenium.remote.http.jdk.JdkHttpClient.execute0(JdkHttpClient.java:500)
	at io.appium.java_client.internal.filters.AppiumIdempotencyFilter.lambda$apply$0(AppiumIdempotencyFilter.java:35)
	at io.appium.java_client.internal.filters.AppiumUserAgentFilter.lambda$apply$0(AppiumUserAgentFilter.java:86)
	at org.openqa.selenium.remote.http.Filter.lambda$andFinally$1(Filter.java:55)
	at org.openqa.selenium.remote.http.jdk.JdkHttpClient.lambda$executeAsync$2(JdkHttpClient.java:387)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
	... 4 more
Caused by: java.net.ConnectException
	at java.net.http/jdk.internal.net.http.HttpClientImpl.send(HttpClientImpl.java:574)
	at java.net.http/jdk.internal.net.http.HttpClientFacade.send(HttpClientFacade.java:123)
	at org.openqa.selenium.remote.http.jdk.JdkHttpClient.execute0(JdkHttpClient.java:459)
	... 9 more
Caused by: java.net.ConnectException
	at java.net.http/jdk.internal.net.http.common.Utils.toConnectException(Utils.java:1055)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.connectAsync(PlainHttpConnection.java:198)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.checkRetryConnect(PlainHttpConnection.java:230)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.lambda$connectAsync$1(PlainHttpConnection.java:206)
	at java.base/java.util.concurrent.CompletableFuture.uniHandle(CompletableFuture.java:934)
	at java.base/java.util.concurrent.CompletableFuture$UniHandle.tryFire(CompletableFuture.java:911)
	at java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1773)
	... 3 more
Caused by: java.nio.channels.ClosedChannelException
	at java.base/sun.nio.ch.SocketChannelImpl.ensureOpen(SocketChannelImpl.java:195)
	at java.base/sun.nio.ch.SocketChannelImpl.beginConnect(SocketChannelImpl.java:760)
	at java.base/sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:848)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.lambda$connectAsync$0(PlainHttpConnection.java:183)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:569)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.connectAsync(PlainHttpConnection.java:185)
	... 9 more
... Removed 21 stack frames</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p>
<h3 id="m8">com.briskpe.tests.LoginTest#tearDown</h3><table class="result"><tr class="param"><th>Parameter #1</th></tr><tr class="param stripe"><td>[TestResult name=shouldNotLoginWithInvalidMobileNumber status=SKIP method=LoginTest.shouldNotLoginWithInvalidMobileNumber()[pri:0, instance:com.briskpe.tests.LoginTest@2f953efd] output={null}]</td></tr><tr><th>Exception</th></tr><tr><td><div class="stacktrace">java.lang.NullPointerException: Cannot invoke &quot;com.aventstack.extentreports.ExtentTest.skip(String)&quot; because &quot;com.briskpe.tests.BaseTest.test&quot; is null
	at com.briskpe.tests.BaseTest.tearDown(BaseTest.java:64)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:842)
... Removed 22 stack frames</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p>
<h3 id="m9">com.briskpe.tests.LoginTest#tearDown</h3><table class="result"><tr class="param"><th>Parameter #1</th></tr><tr class="param stripe"><td>[TestResult name=shouldLoginWithValidMobile status=SKIP method=LoginTest.shouldLoginWithValidMobile()[pri:0, instance:com.briskpe.tests.LoginTest@2f953efd] output={null}]</td></tr><tr><th>Exception</th></tr><tr><td><div class="stacktrace">java.lang.NullPointerException: Cannot invoke &quot;com.aventstack.extentreports.ExtentTest.skip(String)&quot; because &quot;com.briskpe.tests.BaseTest.test&quot; is null
	at com.briskpe.tests.BaseTest.tearDown(BaseTest.java:64)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:842)
... Removed 22 stack frames</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p>
<h3 id="m10">com.briskpe.tests.LoginTest#tearDownLoginTest</h3><table class="result"><tr class="param"><th>Parameter #1</th></tr><tr class="param stripe"><td>[TestResult name=shouldLoginWithValidMobile status=SKIP method=LoginTest.shouldLoginWithValidMobile()[pri:0, instance:com.briskpe.tests.LoginTest@2f953efd] output={null}]</td></tr><tr><th>Exception</th></tr><tr><td><div class="stacktrace">java.lang.NullPointerException: Cannot invoke &quot;com.aventstack.extentreports.ExtentTest.skip(String)&quot; because &quot;com.briskpe.tests.LoginTest.test&quot; is null
	at com.briskpe.tests.LoginTest.tearDownLoginTest(LoginTest.java:71)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:842)
... Removed 22 stack frames</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p>
<h3 id="m11">com.briskpe.tests.LoginTest#tearDownLoginTest</h3><table class="result"><tr class="param"><th>Parameter #1</th></tr><tr class="param stripe"><td>[TestResult name=shouldNotLoginWithInvalidMobileNumber status=SKIP method=LoginTest.shouldNotLoginWithInvalidMobileNumber()[pri:0, instance:com.briskpe.tests.LoginTest@2f953efd] output={null}]</td></tr><tr><th>Exception</th></tr><tr><td><div class="stacktrace">java.lang.NullPointerException: Cannot invoke &quot;com.aventstack.extentreports.ExtentTest.skip(String)&quot; because &quot;com.briskpe.tests.LoginTest.test&quot; is null
	at com.briskpe.tests.LoginTest.tearDownLoginTest(LoginTest.java:71)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:842)
... Removed 22 stack frames</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p>
<h3 id="m12">com.briskpe.tests.DashBoardTest#flushReport</h3><table class="result"><tr><th class="invisible"/></tr></table><p class="totop"><a href="#summary">back to summary</a></p>
<h3 id="m13">com.briskpe.tests.LoginTest#flushLoginReport</h3><table class="result"><tr><th class="invisible"/></tr></table><p class="totop"><a href="#summary">back to summary</a></p>
<h3 id="m14">com.briskpe.tests.DashBoardTest#clickSkipButtonTest</h3><table class="result"><tr><th>Exception</th></tr><tr><td><div class="stacktrace">org.openqa.selenium.SessionNotCreatedException: Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure. 
Host info: host: &apos;VISHALSHUKLA&apos;, ip: &apos;10.188.60.117&apos;
Build info: version: &apos;4.34.0&apos;, revision: &apos;707dcb4246*&apos;
System info: os.name: &apos;Windows 11&apos;, os.arch: &apos;amd64&apos;, os.version: &apos;10.0&apos;, java.version: &apos;17.0.12&apos;
Driver info: io.appium.java_client.ios.IOSDriver
Command: [null, newSession {capabilities=[Capabilities {appium:automationName: Flutter, appium:bundleId: com.briskpe.app, platformName: IOS}]}]
Capabilities {appium:automationName: Flutter, appium:bundleId: com.briskpe.app, platformName: IOS}
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:562)
	at io.appium.java_client.AppiumDriver.startSession(AppiumDriver.java:313)
	at org.openqa.selenium.remote.RemoteWebDriver.&lt;init&gt;(RemoteWebDriver.java:173)
	at io.appium.java_client.AppiumDriver.&lt;init&gt;(AppiumDriver.java:101)
	at io.appium.java_client.AppiumDriver.&lt;init&gt;(AppiumDriver.java:113)
	at io.appium.java_client.ios.IOSDriver.&lt;init&gt;(IOSDriver.java:101)
	at com.briskpe.framework.core.DriverFactory.createIOSDriver(DriverFactory.java:120)
	at com.briskpe.framework.core.DriverFactory.createDriver(DriverFactory.java:52)
	at com.briskpe.framework.core.DriverFactory.createDriver(DriverFactory.java:40)
	at com.briskpe.tests.BaseTest.setUp(BaseTest.java:39)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:842)
Caused by: java.io.UncheckedIOException: java.net.ConnectException
	at org.openqa.selenium.remote.http.jdk.JdkHttpClient.execute0(JdkHttpClient.java:500)
	at io.appium.java_client.internal.filters.AppiumIdempotencyFilter.lambda$apply$0(AppiumIdempotencyFilter.java:35)
	at io.appium.java_client.internal.filters.AppiumUserAgentFilter.lambda$apply$0(AppiumUserAgentFilter.java:86)
	at org.openqa.selenium.remote.http.Filter.lambda$andFinally$1(Filter.java:55)
	at org.openqa.selenium.remote.http.jdk.JdkHttpClient.lambda$executeAsync$2(JdkHttpClient.java:387)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
	... 4 more
Caused by: java.net.ConnectException
	at java.net.http/jdk.internal.net.http.HttpClientImpl.send(HttpClientImpl.java:574)
	at java.net.http/jdk.internal.net.http.HttpClientFacade.send(HttpClientFacade.java:123)
	at org.openqa.selenium.remote.http.jdk.JdkHttpClient.execute0(JdkHttpClient.java:459)
	... 9 more
Caused by: java.net.ConnectException
	at java.net.http/jdk.internal.net.http.common.Utils.toConnectException(Utils.java:1055)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.connectAsync(PlainHttpConnection.java:198)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.checkRetryConnect(PlainHttpConnection.java:230)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.lambda$connectAsync$1(PlainHttpConnection.java:206)
	at java.base/java.util.concurrent.CompletableFuture.uniHandle(CompletableFuture.java:934)
	at java.base/java.util.concurrent.CompletableFuture$UniHandle.tryFire(CompletableFuture.java:911)
	at java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1773)
	... 3 more
Caused by: java.nio.channels.ClosedChannelException
	at java.base/sun.nio.ch.SocketChannelImpl.ensureOpen(SocketChannelImpl.java:195)
	at java.base/sun.nio.ch.SocketChannelImpl.beginConnect(SocketChannelImpl.java:760)
	at java.base/sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:848)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.lambda$connectAsync$0(PlainHttpConnection.java:183)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:569)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.connectAsync(PlainHttpConnection.java:185)
	... 9 more
... Removed 21 stack frames</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p>
<h3 id="m15">com.briskpe.tests.DashBoardTest#verifyUserCanNavigateToSideMenuItems</h3><table class="result"><tr><th>Exception</th></tr><tr><td><div class="stacktrace">java.lang.NullPointerException: Cannot invoke &quot;com.aventstack.extentreports.ExtentTest.skip(String)&quot; because &quot;com.briskpe.tests.DashBoardTest.test&quot; is null
	at com.briskpe.tests.DashBoardTest.tearDownTestWithScreenshot(DashBoardTest.java:89)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:842)
... Removed 22 stack frames</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p>
<h3 id="m16">com.briskpe.tests.LoginTest#shouldLoginWithValidMobile</h3><table class="result"><tr><th>Exception</th></tr><tr><td><div class="stacktrace">org.openqa.selenium.SessionNotCreatedException: Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure. 
Host info: host: &apos;VISHALSHUKLA&apos;, ip: &apos;10.188.60.117&apos;
Build info: version: &apos;4.34.0&apos;, revision: &apos;707dcb4246*&apos;
System info: os.name: &apos;Windows 11&apos;, os.arch: &apos;amd64&apos;, os.version: &apos;10.0&apos;, java.version: &apos;17.0.12&apos;
Driver info: io.appium.java_client.ios.IOSDriver
Command: [null, newSession {capabilities=[Capabilities {appium:automationName: Flutter, appium:bundleId: com.briskpe.app, platformName: IOS}]}]
Capabilities {appium:automationName: Flutter, appium:bundleId: com.briskpe.app, platformName: IOS}
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:562)
	at io.appium.java_client.AppiumDriver.startSession(AppiumDriver.java:313)
	at org.openqa.selenium.remote.RemoteWebDriver.&lt;init&gt;(RemoteWebDriver.java:173)
	at io.appium.java_client.AppiumDriver.&lt;init&gt;(AppiumDriver.java:101)
	at io.appium.java_client.AppiumDriver.&lt;init&gt;(AppiumDriver.java:113)
	at io.appium.java_client.ios.IOSDriver.&lt;init&gt;(IOSDriver.java:101)
	at com.briskpe.framework.core.DriverFactory.createIOSDriver(DriverFactory.java:120)
	at com.briskpe.framework.core.DriverFactory.createDriver(DriverFactory.java:52)
	at com.briskpe.framework.core.DriverFactory.createDriver(DriverFactory.java:40)
	at com.briskpe.tests.BaseTest.setUp(BaseTest.java:39)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:842)
Caused by: java.io.UncheckedIOException: java.net.ConnectException
	at org.openqa.selenium.remote.http.jdk.JdkHttpClient.execute0(JdkHttpClient.java:500)
	at io.appium.java_client.internal.filters.AppiumIdempotencyFilter.lambda$apply$0(AppiumIdempotencyFilter.java:35)
	at io.appium.java_client.internal.filters.AppiumUserAgentFilter.lambda$apply$0(AppiumUserAgentFilter.java:86)
	at org.openqa.selenium.remote.http.Filter.lambda$andFinally$1(Filter.java:55)
	at org.openqa.selenium.remote.http.jdk.JdkHttpClient.lambda$executeAsync$2(JdkHttpClient.java:387)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
	... 4 more
Caused by: java.net.ConnectException
	at java.net.http/jdk.internal.net.http.HttpClientImpl.send(HttpClientImpl.java:574)
	at java.net.http/jdk.internal.net.http.HttpClientFacade.send(HttpClientFacade.java:123)
	at org.openqa.selenium.remote.http.jdk.JdkHttpClient.execute0(JdkHttpClient.java:459)
	... 9 more
Caused by: java.net.ConnectException
	at java.net.http/jdk.internal.net.http.common.Utils.toConnectException(Utils.java:1055)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.connectAsync(PlainHttpConnection.java:198)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.checkRetryConnect(PlainHttpConnection.java:230)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.lambda$connectAsync$1(PlainHttpConnection.java:206)
	at java.base/java.util.concurrent.CompletableFuture.uniHandle(CompletableFuture.java:934)
	at java.base/java.util.concurrent.CompletableFuture$UniHandle.tryFire(CompletableFuture.java:911)
	at java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1773)
	... 3 more
Caused by: java.nio.channels.ClosedChannelException
	at java.base/sun.nio.ch.SocketChannelImpl.ensureOpen(SocketChannelImpl.java:195)
	at java.base/sun.nio.ch.SocketChannelImpl.beginConnect(SocketChannelImpl.java:760)
	at java.base/sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:848)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.lambda$connectAsync$0(PlainHttpConnection.java:183)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:569)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.connectAsync(PlainHttpConnection.java:185)
	... 9 more
... Removed 21 stack frames</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p>
<h3 id="m17">com.briskpe.tests.LoginTest#shouldNotLoginWithInvalidMobileNumber</h3><table class="result"><tr><th>Exception</th></tr><tr><td><div class="stacktrace">java.lang.NullPointerException: Cannot invoke &quot;com.aventstack.extentreports.ExtentTest.skip(String)&quot; because &quot;com.briskpe.tests.LoginTest.test&quot; is null
	at com.briskpe.tests.LoginTest.tearDownLoginTest(LoginTest.java:71)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:842)
... Removed 22 stack frames</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p>
<h2>Android</h2><h3 id="m18">com.briskpe.tests.DashBoardTest#setUp</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th></tr><tr class="param stripe"><td>public void com.briskpe.tests.DashBoardTest.verifyUserCanNavigateToSideMenuItems()</td><td>IOS</td></tr><tr><th colspan="2">Exception</th></tr><tr><td colspan="2"><div class="stacktrace">org.openqa.selenium.SessionNotCreatedException: Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure. 
Host info: host: &apos;VISHALSHUKLA&apos;, ip: &apos;10.188.60.117&apos;
Build info: version: &apos;4.34.0&apos;, revision: &apos;707dcb4246*&apos;
System info: os.name: &apos;Windows 11&apos;, os.arch: &apos;amd64&apos;, os.version: &apos;10.0&apos;, java.version: &apos;17.0.12&apos;
Driver info: io.appium.java_client.ios.IOSDriver
Command: [null, newSession {capabilities=[Capabilities {appium:automationName: Flutter, appium:bundleId: com.briskpe.app, platformName: IOS}]}]
Capabilities {appium:automationName: Flutter, appium:bundleId: com.briskpe.app, platformName: IOS}
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:562)
	at io.appium.java_client.AppiumDriver.startSession(AppiumDriver.java:313)
	at org.openqa.selenium.remote.RemoteWebDriver.&lt;init&gt;(RemoteWebDriver.java:173)
	at io.appium.java_client.AppiumDriver.&lt;init&gt;(AppiumDriver.java:101)
	at io.appium.java_client.AppiumDriver.&lt;init&gt;(AppiumDriver.java:113)
	at io.appium.java_client.ios.IOSDriver.&lt;init&gt;(IOSDriver.java:101)
	at com.briskpe.framework.core.DriverFactory.createIOSDriver(DriverFactory.java:120)
	at com.briskpe.framework.core.DriverFactory.createDriver(DriverFactory.java:52)
	at com.briskpe.framework.core.DriverFactory.createDriver(DriverFactory.java:40)
	at com.briskpe.tests.BaseTest.setUp(BaseTest.java:39)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:842)
Caused by: java.io.UncheckedIOException: java.net.ConnectException
	at org.openqa.selenium.remote.http.jdk.JdkHttpClient.execute0(JdkHttpClient.java:500)
	at io.appium.java_client.internal.filters.AppiumIdempotencyFilter.lambda$apply$0(AppiumIdempotencyFilter.java:35)
	at io.appium.java_client.internal.filters.AppiumUserAgentFilter.lambda$apply$0(AppiumUserAgentFilter.java:86)
	at org.openqa.selenium.remote.http.Filter.lambda$andFinally$1(Filter.java:55)
	at org.openqa.selenium.remote.http.jdk.JdkHttpClient.lambda$executeAsync$2(JdkHttpClient.java:387)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
	... 4 more
Caused by: java.net.ConnectException
	at java.net.http/jdk.internal.net.http.HttpClientImpl.send(HttpClientImpl.java:574)
	at java.net.http/jdk.internal.net.http.HttpClientFacade.send(HttpClientFacade.java:123)
	at org.openqa.selenium.remote.http.jdk.JdkHttpClient.execute0(JdkHttpClient.java:459)
	... 9 more
Caused by: java.net.ConnectException
	at java.net.http/jdk.internal.net.http.common.Utils.toConnectException(Utils.java:1055)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.connectAsync(PlainHttpConnection.java:198)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.checkRetryConnect(PlainHttpConnection.java:230)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.lambda$connectAsync$1(PlainHttpConnection.java:206)
	at java.base/java.util.concurrent.CompletableFuture.uniHandle(CompletableFuture.java:934)
	at java.base/java.util.concurrent.CompletableFuture$UniHandle.tryFire(CompletableFuture.java:911)
	at java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1773)
	... 3 more
Caused by: java.nio.channels.ClosedChannelException
	at java.base/sun.nio.ch.SocketChannelImpl.ensureOpen(SocketChannelImpl.java:195)
	at java.base/sun.nio.ch.SocketChannelImpl.beginConnect(SocketChannelImpl.java:760)
	at java.base/sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:848)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.lambda$connectAsync$0(PlainHttpConnection.java:183)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:569)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.connectAsync(PlainHttpConnection.java:185)
	... 9 more
... Removed 21 stack frames</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p>
<h3 id="m19">com.briskpe.tests.DashBoardTest#setUp</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th></tr><tr class="param stripe"><td>public void com.briskpe.tests.DashBoardTest.clickSkipButtonTest()</td><td>IOS</td></tr><tr><th colspan="2">Exception</th></tr><tr><td colspan="2"><div class="stacktrace">org.openqa.selenium.SessionNotCreatedException: Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure. 
Host info: host: &apos;VISHALSHUKLA&apos;, ip: &apos;10.188.60.117&apos;
Build info: version: &apos;4.34.0&apos;, revision: &apos;707dcb4246*&apos;
System info: os.name: &apos;Windows 11&apos;, os.arch: &apos;amd64&apos;, os.version: &apos;10.0&apos;, java.version: &apos;17.0.12&apos;
Driver info: io.appium.java_client.ios.IOSDriver
Command: [null, newSession {capabilities=[Capabilities {appium:automationName: Flutter, appium:bundleId: com.briskpe.app, platformName: IOS}]}]
Capabilities {appium:automationName: Flutter, appium:bundleId: com.briskpe.app, platformName: IOS}
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:562)
	at io.appium.java_client.AppiumDriver.startSession(AppiumDriver.java:313)
	at org.openqa.selenium.remote.RemoteWebDriver.&lt;init&gt;(RemoteWebDriver.java:173)
	at io.appium.java_client.AppiumDriver.&lt;init&gt;(AppiumDriver.java:101)
	at io.appium.java_client.AppiumDriver.&lt;init&gt;(AppiumDriver.java:113)
	at io.appium.java_client.ios.IOSDriver.&lt;init&gt;(IOSDriver.java:101)
	at com.briskpe.framework.core.DriverFactory.createIOSDriver(DriverFactory.java:120)
	at com.briskpe.framework.core.DriverFactory.createDriver(DriverFactory.java:52)
	at com.briskpe.framework.core.DriverFactory.createDriver(DriverFactory.java:40)
	at com.briskpe.tests.BaseTest.setUp(BaseTest.java:39)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:842)
Caused by: java.io.UncheckedIOException: java.net.ConnectException
	at org.openqa.selenium.remote.http.jdk.JdkHttpClient.execute0(JdkHttpClient.java:500)
	at io.appium.java_client.internal.filters.AppiumIdempotencyFilter.lambda$apply$0(AppiumIdempotencyFilter.java:35)
	at io.appium.java_client.internal.filters.AppiumUserAgentFilter.lambda$apply$0(AppiumUserAgentFilter.java:86)
	at org.openqa.selenium.remote.http.Filter.lambda$andFinally$1(Filter.java:55)
	at org.openqa.selenium.remote.http.jdk.JdkHttpClient.lambda$executeAsync$2(JdkHttpClient.java:387)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
	... 4 more
Caused by: java.net.ConnectException
	at java.net.http/jdk.internal.net.http.HttpClientImpl.send(HttpClientImpl.java:574)
	at java.net.http/jdk.internal.net.http.HttpClientFacade.send(HttpClientFacade.java:123)
	at org.openqa.selenium.remote.http.jdk.JdkHttpClient.execute0(JdkHttpClient.java:459)
	... 9 more
Caused by: java.net.ConnectException
	at java.net.http/jdk.internal.net.http.common.Utils.toConnectException(Utils.java:1055)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.connectAsync(PlainHttpConnection.java:198)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.checkRetryConnect(PlainHttpConnection.java:230)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.lambda$connectAsync$1(PlainHttpConnection.java:206)
	at java.base/java.util.concurrent.CompletableFuture.uniHandle(CompletableFuture.java:934)
	at java.base/java.util.concurrent.CompletableFuture$UniHandle.tryFire(CompletableFuture.java:911)
	at java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1773)
	... 3 more
Caused by: java.nio.channels.ClosedChannelException
	at java.base/sun.nio.ch.SocketChannelImpl.ensureOpen(SocketChannelImpl.java:195)
	at java.base/sun.nio.ch.SocketChannelImpl.beginConnect(SocketChannelImpl.java:760)
	at java.base/sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:848)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.lambda$connectAsync$0(PlainHttpConnection.java:183)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:569)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.connectAsync(PlainHttpConnection.java:185)
	... 9 more
... Removed 21 stack frames</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p>
<h3 id="m20">com.briskpe.tests.DashBoardTest#tearDown</h3><table class="result"><tr class="param"><th>Parameter #1</th></tr><tr class="param stripe"><td>[TestResult name=verifyUserCanNavigateToSideMenuItems status=SKIP method=DashBoardTest.verifyUserCanNavigateToSideMenuItems()[pri:2, instance:com.briskpe.tests.DashBoardTest@2254127a] output={null}]</td></tr><tr><th>Exception</th></tr><tr><td><div class="stacktrace">java.lang.NullPointerException: Cannot invoke &quot;com.aventstack.extentreports.ExtentTest.skip(String)&quot; because &quot;com.briskpe.tests.BaseTest.test&quot; is null
	at com.briskpe.tests.BaseTest.tearDown(BaseTest.java:64)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:842)
... Removed 22 stack frames</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p>
<h3 id="m21">com.briskpe.tests.DashBoardTest#tearDown</h3><table class="result"><tr class="param"><th>Parameter #1</th></tr><tr class="param stripe"><td>[TestResult name=clickSkipButtonTest status=SKIP method=DashBoardTest.clickSkipButtonTest()[pri:1, instance:com.briskpe.tests.DashBoardTest@2254127a] output={null}]</td></tr><tr><th>Exception</th></tr><tr><td><div class="stacktrace">java.lang.NullPointerException: Cannot invoke &quot;com.aventstack.extentreports.ExtentTest.skip(String)&quot; because &quot;com.briskpe.tests.BaseTest.test&quot; is null
	at com.briskpe.tests.BaseTest.tearDown(BaseTest.java:64)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:842)
... Removed 22 stack frames</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p>
<h3 id="m22">com.briskpe.tests.DashBoardTest#tearDownTestWithScreenshot</h3><table class="result"><tr class="param"><th>Parameter #1</th></tr><tr class="param stripe"><td>[TestResult name=clickSkipButtonTest status=SKIP method=DashBoardTest.clickSkipButtonTest()[pri:1, instance:com.briskpe.tests.DashBoardTest@2254127a] output={null}]</td></tr><tr><th>Exception</th></tr><tr><td><div class="stacktrace">java.lang.NullPointerException: Cannot invoke &quot;com.aventstack.extentreports.ExtentTest.skip(String)&quot; because &quot;com.briskpe.tests.DashBoardTest.test&quot; is null
	at com.briskpe.tests.DashBoardTest.tearDownTestWithScreenshot(DashBoardTest.java:89)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:842)
... Removed 22 stack frames</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p>
<h3 id="m23">com.briskpe.tests.DashBoardTest#tearDownTestWithScreenshot</h3><table class="result"><tr class="param"><th>Parameter #1</th></tr><tr class="param stripe"><td>[TestResult name=verifyUserCanNavigateToSideMenuItems status=SKIP method=DashBoardTest.verifyUserCanNavigateToSideMenuItems()[pri:2, instance:com.briskpe.tests.DashBoardTest@2254127a] output={null}]</td></tr><tr><th>Exception</th></tr><tr><td><div class="stacktrace">java.lang.NullPointerException: Cannot invoke &quot;com.aventstack.extentreports.ExtentTest.skip(String)&quot; because &quot;com.briskpe.tests.DashBoardTest.test&quot; is null
	at com.briskpe.tests.DashBoardTest.tearDownTestWithScreenshot(DashBoardTest.java:89)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:842)
... Removed 22 stack frames</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p>
<h3 id="m24">com.briskpe.tests.LoginTest#setUp</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th></tr><tr class="param stripe"><td>public void com.briskpe.tests.LoginTest.shouldLoginWithValidMobile()</td><td>ANDROID</td></tr><tr><th colspan="2">Exception</th></tr><tr><td colspan="2"><div class="stacktrace">org.openqa.selenium.SessionNotCreatedException: Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure. 
Host info: host: &apos;VISHALSHUKLA&apos;, ip: &apos;10.188.60.117&apos;
Build info: version: &apos;4.34.0&apos;, revision: &apos;707dcb4246*&apos;
System info: os.name: &apos;Windows 11&apos;, os.arch: &apos;amd64&apos;, os.version: &apos;10.0&apos;, java.version: &apos;17.0.12&apos;
Driver info: io.appium.java_client.ios.IOSDriver
Command: [null, newSession {capabilities=[Capabilities {appium:automationName: Flutter, appium:bundleId: com.briskpe.app, platformName: IOS}]}]
Capabilities {appium:automationName: Flutter, appium:bundleId: com.briskpe.app, platformName: IOS}
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:562)
	at io.appium.java_client.AppiumDriver.startSession(AppiumDriver.java:313)
	at org.openqa.selenium.remote.RemoteWebDriver.&lt;init&gt;(RemoteWebDriver.java:173)
	at io.appium.java_client.AppiumDriver.&lt;init&gt;(AppiumDriver.java:101)
	at io.appium.java_client.AppiumDriver.&lt;init&gt;(AppiumDriver.java:113)
	at io.appium.java_client.ios.IOSDriver.&lt;init&gt;(IOSDriver.java:101)
	at com.briskpe.framework.core.DriverFactory.createIOSDriver(DriverFactory.java:120)
	at com.briskpe.framework.core.DriverFactory.createDriver(DriverFactory.java:52)
	at com.briskpe.framework.core.DriverFactory.createDriver(DriverFactory.java:40)
	at com.briskpe.tests.BaseTest.setUp(BaseTest.java:39)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:842)
Caused by: java.io.UncheckedIOException: java.net.ConnectException
	at org.openqa.selenium.remote.http.jdk.JdkHttpClient.execute0(JdkHttpClient.java:500)
	at io.appium.java_client.internal.filters.AppiumIdempotencyFilter.lambda$apply$0(AppiumIdempotencyFilter.java:35)
	at io.appium.java_client.internal.filters.AppiumUserAgentFilter.lambda$apply$0(AppiumUserAgentFilter.java:86)
	at org.openqa.selenium.remote.http.Filter.lambda$andFinally$1(Filter.java:55)
	at org.openqa.selenium.remote.http.jdk.JdkHttpClient.lambda$executeAsync$2(JdkHttpClient.java:387)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
	... 4 more
Caused by: java.net.ConnectException
	at java.net.http/jdk.internal.net.http.HttpClientImpl.send(HttpClientImpl.java:574)
	at java.net.http/jdk.internal.net.http.HttpClientFacade.send(HttpClientFacade.java:123)
	at org.openqa.selenium.remote.http.jdk.JdkHttpClient.execute0(JdkHttpClient.java:459)
	... 9 more
Caused by: java.net.ConnectException
	at java.net.http/jdk.internal.net.http.common.Utils.toConnectException(Utils.java:1055)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.connectAsync(PlainHttpConnection.java:198)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.checkRetryConnect(PlainHttpConnection.java:230)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.lambda$connectAsync$1(PlainHttpConnection.java:206)
	at java.base/java.util.concurrent.CompletableFuture.uniHandle(CompletableFuture.java:934)
	at java.base/java.util.concurrent.CompletableFuture$UniHandle.tryFire(CompletableFuture.java:911)
	at java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1773)
	... 3 more
Caused by: java.nio.channels.ClosedChannelException
	at java.base/sun.nio.ch.SocketChannelImpl.ensureOpen(SocketChannelImpl.java:195)
	at java.base/sun.nio.ch.SocketChannelImpl.beginConnect(SocketChannelImpl.java:760)
	at java.base/sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:848)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.lambda$connectAsync$0(PlainHttpConnection.java:183)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:569)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.connectAsync(PlainHttpConnection.java:185)
	... 9 more
... Removed 21 stack frames</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p>
<h3 id="m25">com.briskpe.tests.LoginTest#setUp</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th></tr><tr class="param stripe"><td>public void com.briskpe.tests.LoginTest.shouldNotLoginWithInvalidMobileNumber()</td><td>IOS</td></tr><tr><th colspan="2">Exception</th></tr><tr><td colspan="2"><div class="stacktrace">org.openqa.selenium.SessionNotCreatedException: Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure. 
Host info: host: &apos;VISHALSHUKLA&apos;, ip: &apos;10.188.60.117&apos;
Build info: version: &apos;4.34.0&apos;, revision: &apos;707dcb4246*&apos;
System info: os.name: &apos;Windows 11&apos;, os.arch: &apos;amd64&apos;, os.version: &apos;10.0&apos;, java.version: &apos;17.0.12&apos;
Driver info: io.appium.java_client.ios.IOSDriver
Command: [null, newSession {capabilities=[Capabilities {appium:automationName: Flutter, appium:bundleId: com.briskpe.app, platformName: IOS}]}]
Capabilities {appium:automationName: Flutter, appium:bundleId: com.briskpe.app, platformName: IOS}
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:562)
	at io.appium.java_client.AppiumDriver.startSession(AppiumDriver.java:313)
	at org.openqa.selenium.remote.RemoteWebDriver.&lt;init&gt;(RemoteWebDriver.java:173)
	at io.appium.java_client.AppiumDriver.&lt;init&gt;(AppiumDriver.java:101)
	at io.appium.java_client.AppiumDriver.&lt;init&gt;(AppiumDriver.java:113)
	at io.appium.java_client.ios.IOSDriver.&lt;init&gt;(IOSDriver.java:101)
	at com.briskpe.framework.core.DriverFactory.createIOSDriver(DriverFactory.java:120)
	at com.briskpe.framework.core.DriverFactory.createDriver(DriverFactory.java:52)
	at com.briskpe.framework.core.DriverFactory.createDriver(DriverFactory.java:40)
	at com.briskpe.tests.BaseTest.setUp(BaseTest.java:39)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:842)
Caused by: java.io.UncheckedIOException: java.net.ConnectException
	at org.openqa.selenium.remote.http.jdk.JdkHttpClient.execute0(JdkHttpClient.java:500)
	at io.appium.java_client.internal.filters.AppiumIdempotencyFilter.lambda$apply$0(AppiumIdempotencyFilter.java:35)
	at io.appium.java_client.internal.filters.AppiumUserAgentFilter.lambda$apply$0(AppiumUserAgentFilter.java:86)
	at org.openqa.selenium.remote.http.Filter.lambda$andFinally$1(Filter.java:55)
	at org.openqa.selenium.remote.http.jdk.JdkHttpClient.lambda$executeAsync$2(JdkHttpClient.java:387)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
	... 4 more
Caused by: java.net.ConnectException
	at java.net.http/jdk.internal.net.http.HttpClientImpl.send(HttpClientImpl.java:574)
	at java.net.http/jdk.internal.net.http.HttpClientFacade.send(HttpClientFacade.java:123)
	at org.openqa.selenium.remote.http.jdk.JdkHttpClient.execute0(JdkHttpClient.java:459)
	... 9 more
Caused by: java.net.ConnectException
	at java.net.http/jdk.internal.net.http.common.Utils.toConnectException(Utils.java:1055)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.connectAsync(PlainHttpConnection.java:198)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.checkRetryConnect(PlainHttpConnection.java:230)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.lambda$connectAsync$1(PlainHttpConnection.java:206)
	at java.base/java.util.concurrent.CompletableFuture.uniHandle(CompletableFuture.java:934)
	at java.base/java.util.concurrent.CompletableFuture$UniHandle.tryFire(CompletableFuture.java:911)
	at java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1773)
	... 3 more
Caused by: java.nio.channels.ClosedChannelException
	at java.base/sun.nio.ch.SocketChannelImpl.ensureOpen(SocketChannelImpl.java:195)
	at java.base/sun.nio.ch.SocketChannelImpl.beginConnect(SocketChannelImpl.java:760)
	at java.base/sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:848)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.lambda$connectAsync$0(PlainHttpConnection.java:183)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:569)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.connectAsync(PlainHttpConnection.java:185)
	... 9 more
... Removed 21 stack frames</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p>
<h3 id="m26">com.briskpe.tests.LoginTest#tearDown</h3><table class="result"><tr class="param"><th>Parameter #1</th></tr><tr class="param stripe"><td>[TestResult name=shouldLoginWithValidMobile status=SKIP method=LoginTest.shouldLoginWithValidMobile()[pri:0, instance:com.briskpe.tests.LoginTest@5dafbe45] output={null}]</td></tr><tr><th>Exception</th></tr><tr><td><div class="stacktrace">java.lang.NullPointerException: Cannot invoke &quot;com.aventstack.extentreports.ExtentTest.skip(String)&quot; because &quot;com.briskpe.tests.BaseTest.test&quot; is null
	at com.briskpe.tests.BaseTest.tearDown(BaseTest.java:64)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:842)
... Removed 22 stack frames</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p>
<h3 id="m27">com.briskpe.tests.LoginTest#tearDown</h3><table class="result"><tr class="param"><th>Parameter #1</th></tr><tr class="param stripe"><td>[TestResult name=shouldNotLoginWithInvalidMobileNumber status=SKIP method=LoginTest.shouldNotLoginWithInvalidMobileNumber()[pri:0, instance:com.briskpe.tests.LoginTest@5dafbe45] output={null}]</td></tr><tr><th>Exception</th></tr><tr><td><div class="stacktrace">java.lang.NullPointerException: Cannot invoke &quot;com.aventstack.extentreports.ExtentTest.skip(String)&quot; because &quot;com.briskpe.tests.BaseTest.test&quot; is null
	at com.briskpe.tests.BaseTest.tearDown(BaseTest.java:64)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:842)
... Removed 22 stack frames</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p>
<h3 id="m28">com.briskpe.tests.LoginTest#tearDownLoginTest</h3><table class="result"><tr class="param"><th>Parameter #1</th></tr><tr class="param stripe"><td>[TestResult name=shouldNotLoginWithInvalidMobileNumber status=SKIP method=LoginTest.shouldNotLoginWithInvalidMobileNumber()[pri:0, instance:com.briskpe.tests.LoginTest@5dafbe45] output={null}]</td></tr><tr><th>Exception</th></tr><tr><td><div class="stacktrace">java.lang.NullPointerException: Cannot invoke &quot;com.aventstack.extentreports.ExtentTest.skip(String)&quot; because &quot;com.briskpe.tests.LoginTest.test&quot; is null
	at com.briskpe.tests.LoginTest.tearDownLoginTest(LoginTest.java:71)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:842)
... Removed 22 stack frames</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p>
<h3 id="m29">com.briskpe.tests.LoginTest#tearDownLoginTest</h3><table class="result"><tr class="param"><th>Parameter #1</th></tr><tr class="param stripe"><td>[TestResult name=shouldLoginWithValidMobile status=SKIP method=LoginTest.shouldLoginWithValidMobile()[pri:0, instance:com.briskpe.tests.LoginTest@5dafbe45] output={null}]</td></tr><tr><th>Exception</th></tr><tr><td><div class="stacktrace">java.lang.NullPointerException: Cannot invoke &quot;com.aventstack.extentreports.ExtentTest.skip(String)&quot; because &quot;com.briskpe.tests.LoginTest.test&quot; is null
	at com.briskpe.tests.LoginTest.tearDownLoginTest(LoginTest.java:71)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:842)
... Removed 22 stack frames</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p>
<h3 id="m30">com.briskpe.tests.DashBoardTest#clickSkipButtonTest</h3><table class="result"><tr><th>Exception</th></tr><tr><td><div class="stacktrace">org.openqa.selenium.SessionNotCreatedException: Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure. 
Host info: host: &apos;VISHALSHUKLA&apos;, ip: &apos;10.188.60.117&apos;
Build info: version: &apos;4.34.0&apos;, revision: &apos;707dcb4246*&apos;
System info: os.name: &apos;Windows 11&apos;, os.arch: &apos;amd64&apos;, os.version: &apos;10.0&apos;, java.version: &apos;17.0.12&apos;
Driver info: io.appium.java_client.ios.IOSDriver
Command: [null, newSession {capabilities=[Capabilities {appium:automationName: Flutter, appium:bundleId: com.briskpe.app, platformName: IOS}]}]
Capabilities {appium:automationName: Flutter, appium:bundleId: com.briskpe.app, platformName: IOS}
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:562)
	at io.appium.java_client.AppiumDriver.startSession(AppiumDriver.java:313)
	at org.openqa.selenium.remote.RemoteWebDriver.&lt;init&gt;(RemoteWebDriver.java:173)
	at io.appium.java_client.AppiumDriver.&lt;init&gt;(AppiumDriver.java:101)
	at io.appium.java_client.AppiumDriver.&lt;init&gt;(AppiumDriver.java:113)
	at io.appium.java_client.ios.IOSDriver.&lt;init&gt;(IOSDriver.java:101)
	at com.briskpe.framework.core.DriverFactory.createIOSDriver(DriverFactory.java:120)
	at com.briskpe.framework.core.DriverFactory.createDriver(DriverFactory.java:52)
	at com.briskpe.framework.core.DriverFactory.createDriver(DriverFactory.java:40)
	at com.briskpe.tests.BaseTest.setUp(BaseTest.java:39)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:842)
Caused by: java.io.UncheckedIOException: java.net.ConnectException
	at org.openqa.selenium.remote.http.jdk.JdkHttpClient.execute0(JdkHttpClient.java:500)
	at io.appium.java_client.internal.filters.AppiumIdempotencyFilter.lambda$apply$0(AppiumIdempotencyFilter.java:35)
	at io.appium.java_client.internal.filters.AppiumUserAgentFilter.lambda$apply$0(AppiumUserAgentFilter.java:86)
	at org.openqa.selenium.remote.http.Filter.lambda$andFinally$1(Filter.java:55)
	at org.openqa.selenium.remote.http.jdk.JdkHttpClient.lambda$executeAsync$2(JdkHttpClient.java:387)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
	... 4 more
Caused by: java.net.ConnectException
	at java.net.http/jdk.internal.net.http.HttpClientImpl.send(HttpClientImpl.java:574)
	at java.net.http/jdk.internal.net.http.HttpClientFacade.send(HttpClientFacade.java:123)
	at org.openqa.selenium.remote.http.jdk.JdkHttpClient.execute0(JdkHttpClient.java:459)
	... 9 more
Caused by: java.net.ConnectException
	at java.net.http/jdk.internal.net.http.common.Utils.toConnectException(Utils.java:1055)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.connectAsync(PlainHttpConnection.java:198)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.checkRetryConnect(PlainHttpConnection.java:230)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.lambda$connectAsync$1(PlainHttpConnection.java:206)
	at java.base/java.util.concurrent.CompletableFuture.uniHandle(CompletableFuture.java:934)
	at java.base/java.util.concurrent.CompletableFuture$UniHandle.tryFire(CompletableFuture.java:911)
	at java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1773)
	... 3 more
Caused by: java.nio.channels.ClosedChannelException
	at java.base/sun.nio.ch.SocketChannelImpl.ensureOpen(SocketChannelImpl.java:195)
	at java.base/sun.nio.ch.SocketChannelImpl.beginConnect(SocketChannelImpl.java:760)
	at java.base/sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:848)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.lambda$connectAsync$0(PlainHttpConnection.java:183)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:569)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.connectAsync(PlainHttpConnection.java:185)
	... 9 more
... Removed 21 stack frames</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p>
<h3 id="m31">com.briskpe.tests.DashBoardTest#verifyUserCanNavigateToSideMenuItems</h3><table class="result"><tr><th>Exception</th></tr><tr><td><div class="stacktrace">java.lang.NullPointerException: Cannot invoke &quot;com.aventstack.extentreports.ExtentTest.skip(String)&quot; because &quot;com.briskpe.tests.BaseTest.test&quot; is null
	at com.briskpe.tests.BaseTest.tearDown(BaseTest.java:64)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:842)
... Removed 22 stack frames</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p>
<h3 id="m32">com.briskpe.tests.LoginTest#shouldLoginWithValidMobile</h3><table class="result"><tr><th>Exception</th></tr><tr><td><div class="stacktrace">org.openqa.selenium.SessionNotCreatedException: Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure. 
Host info: host: &apos;VISHALSHUKLA&apos;, ip: &apos;10.188.60.117&apos;
Build info: version: &apos;4.34.0&apos;, revision: &apos;707dcb4246*&apos;
System info: os.name: &apos;Windows 11&apos;, os.arch: &apos;amd64&apos;, os.version: &apos;10.0&apos;, java.version: &apos;17.0.12&apos;
Driver info: io.appium.java_client.ios.IOSDriver
Command: [null, newSession {capabilities=[Capabilities {appium:automationName: Flutter, appium:bundleId: com.briskpe.app, platformName: IOS}]}]
Capabilities {appium:automationName: Flutter, appium:bundleId: com.briskpe.app, platformName: IOS}
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:562)
	at io.appium.java_client.AppiumDriver.startSession(AppiumDriver.java:313)
	at org.openqa.selenium.remote.RemoteWebDriver.&lt;init&gt;(RemoteWebDriver.java:173)
	at io.appium.java_client.AppiumDriver.&lt;init&gt;(AppiumDriver.java:101)
	at io.appium.java_client.AppiumDriver.&lt;init&gt;(AppiumDriver.java:113)
	at io.appium.java_client.ios.IOSDriver.&lt;init&gt;(IOSDriver.java:101)
	at com.briskpe.framework.core.DriverFactory.createIOSDriver(DriverFactory.java:120)
	at com.briskpe.framework.core.DriverFactory.createDriver(DriverFactory.java:52)
	at com.briskpe.framework.core.DriverFactory.createDriver(DriverFactory.java:40)
	at com.briskpe.tests.BaseTest.setUp(BaseTest.java:39)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:842)
Caused by: java.io.UncheckedIOException: java.net.ConnectException
	at org.openqa.selenium.remote.http.jdk.JdkHttpClient.execute0(JdkHttpClient.java:500)
	at io.appium.java_client.internal.filters.AppiumIdempotencyFilter.lambda$apply$0(AppiumIdempotencyFilter.java:35)
	at io.appium.java_client.internal.filters.AppiumUserAgentFilter.lambda$apply$0(AppiumUserAgentFilter.java:86)
	at org.openqa.selenium.remote.http.Filter.lambda$andFinally$1(Filter.java:55)
	at org.openqa.selenium.remote.http.jdk.JdkHttpClient.lambda$executeAsync$2(JdkHttpClient.java:387)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
	... 4 more
Caused by: java.net.ConnectException
	at java.net.http/jdk.internal.net.http.HttpClientImpl.send(HttpClientImpl.java:574)
	at java.net.http/jdk.internal.net.http.HttpClientFacade.send(HttpClientFacade.java:123)
	at org.openqa.selenium.remote.http.jdk.JdkHttpClient.execute0(JdkHttpClient.java:459)
	... 9 more
Caused by: java.net.ConnectException
	at java.net.http/jdk.internal.net.http.common.Utils.toConnectException(Utils.java:1055)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.connectAsync(PlainHttpConnection.java:198)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.checkRetryConnect(PlainHttpConnection.java:230)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.lambda$connectAsync$1(PlainHttpConnection.java:206)
	at java.base/java.util.concurrent.CompletableFuture.uniHandle(CompletableFuture.java:934)
	at java.base/java.util.concurrent.CompletableFuture$UniHandle.tryFire(CompletableFuture.java:911)
	at java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1773)
	... 3 more
Caused by: java.nio.channels.ClosedChannelException
	at java.base/sun.nio.ch.SocketChannelImpl.ensureOpen(SocketChannelImpl.java:195)
	at java.base/sun.nio.ch.SocketChannelImpl.beginConnect(SocketChannelImpl.java:760)
	at java.base/sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:848)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.lambda$connectAsync$0(PlainHttpConnection.java:183)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:569)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.connectAsync(PlainHttpConnection.java:185)
	... 9 more
... Removed 21 stack frames</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p>
<h3 id="m33">com.briskpe.tests.LoginTest#shouldNotLoginWithInvalidMobileNumber</h3><table class="result"><tr><th>Exception</th></tr><tr><td><div class="stacktrace">java.lang.NullPointerException: Cannot invoke &quot;com.aventstack.extentreports.ExtentTest.skip(String)&quot; because &quot;com.briskpe.tests.BaseTest.test&quot; is null
	at com.briskpe.tests.BaseTest.tearDown(BaseTest.java:64)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:842)
... Removed 22 stack frames</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p>
<h2>Web</h2><h3 id="m34">com.briskpe.tests.DashBoardTest#setUp</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th></tr><tr class="param stripe"><td>public void com.briskpe.tests.DashBoardTest.clickSkipButtonTest()</td><td>IOS</td></tr><tr><th colspan="2">Exception</th></tr><tr><td colspan="2"><div class="stacktrace">org.openqa.selenium.SessionNotCreatedException: Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure. 
Host info: host: &apos;VISHALSHUKLA&apos;, ip: &apos;10.188.60.117&apos;
Build info: version: &apos;4.34.0&apos;, revision: &apos;707dcb4246*&apos;
System info: os.name: &apos;Windows 11&apos;, os.arch: &apos;amd64&apos;, os.version: &apos;10.0&apos;, java.version: &apos;17.0.12&apos;
Driver info: io.appium.java_client.ios.IOSDriver
Command: [null, newSession {capabilities=[Capabilities {appium:automationName: Flutter, appium:bundleId: com.briskpe.app, platformName: IOS}]}]
Capabilities {appium:automationName: Flutter, appium:bundleId: com.briskpe.app, platformName: IOS}
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:562)
	at io.appium.java_client.AppiumDriver.startSession(AppiumDriver.java:313)
	at org.openqa.selenium.remote.RemoteWebDriver.&lt;init&gt;(RemoteWebDriver.java:173)
	at io.appium.java_client.AppiumDriver.&lt;init&gt;(AppiumDriver.java:101)
	at io.appium.java_client.AppiumDriver.&lt;init&gt;(AppiumDriver.java:113)
	at io.appium.java_client.ios.IOSDriver.&lt;init&gt;(IOSDriver.java:101)
	at com.briskpe.framework.core.DriverFactory.createIOSDriver(DriverFactory.java:120)
	at com.briskpe.framework.core.DriverFactory.createDriver(DriverFactory.java:52)
	at com.briskpe.framework.core.DriverFactory.createDriver(DriverFactory.java:40)
	at com.briskpe.tests.BaseTest.setUp(BaseTest.java:39)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:842)
Caused by: java.io.UncheckedIOException: java.net.ConnectException
	at org.openqa.selenium.remote.http.jdk.JdkHttpClient.execute0(JdkHttpClient.java:500)
	at io.appium.java_client.internal.filters.AppiumIdempotencyFilter.lambda$apply$0(AppiumIdempotencyFilter.java:35)
	at io.appium.java_client.internal.filters.AppiumUserAgentFilter.lambda$apply$0(AppiumUserAgentFilter.java:86)
	at org.openqa.selenium.remote.http.Filter.lambda$andFinally$1(Filter.java:55)
	at org.openqa.selenium.remote.http.jdk.JdkHttpClient.lambda$executeAsync$2(JdkHttpClient.java:387)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
	... 4 more
Caused by: java.net.ConnectException
	at java.net.http/jdk.internal.net.http.HttpClientImpl.send(HttpClientImpl.java:574)
	at java.net.http/jdk.internal.net.http.HttpClientFacade.send(HttpClientFacade.java:123)
	at org.openqa.selenium.remote.http.jdk.JdkHttpClient.execute0(JdkHttpClient.java:459)
	... 9 more
Caused by: java.net.ConnectException
	at java.net.http/jdk.internal.net.http.common.Utils.toConnectException(Utils.java:1055)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.connectAsync(PlainHttpConnection.java:198)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.checkRetryConnect(PlainHttpConnection.java:230)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.lambda$connectAsync$1(PlainHttpConnection.java:206)
	at java.base/java.util.concurrent.CompletableFuture.uniHandle(CompletableFuture.java:934)
	at java.base/java.util.concurrent.CompletableFuture$UniHandle.tryFire(CompletableFuture.java:911)
	at java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1773)
	... 3 more
Caused by: java.nio.channels.ClosedChannelException
	at java.base/sun.nio.ch.SocketChannelImpl.ensureOpen(SocketChannelImpl.java:195)
	at java.base/sun.nio.ch.SocketChannelImpl.beginConnect(SocketChannelImpl.java:760)
	at java.base/sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:848)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.lambda$connectAsync$0(PlainHttpConnection.java:183)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:569)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.connectAsync(PlainHttpConnection.java:185)
	... 9 more
... Removed 21 stack frames</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p>
<h3 id="m35">com.briskpe.tests.DashBoardTest#setUp</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th></tr><tr class="param stripe"><td>public void com.briskpe.tests.DashBoardTest.verifyUserCanNavigateToSideMenuItems()</td><td>IOS</td></tr><tr><th colspan="2">Exception</th></tr><tr><td colspan="2"><div class="stacktrace">org.openqa.selenium.SessionNotCreatedException: Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure. 
Host info: host: &apos;VISHALSHUKLA&apos;, ip: &apos;10.188.60.117&apos;
Build info: version: &apos;4.34.0&apos;, revision: &apos;707dcb4246*&apos;
System info: os.name: &apos;Windows 11&apos;, os.arch: &apos;amd64&apos;, os.version: &apos;10.0&apos;, java.version: &apos;17.0.12&apos;
Driver info: io.appium.java_client.ios.IOSDriver
Command: [null, newSession {capabilities=[Capabilities {appium:automationName: Flutter, appium:bundleId: com.briskpe.app, platformName: IOS}]}]
Capabilities {appium:automationName: Flutter, appium:bundleId: com.briskpe.app, platformName: IOS}
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:562)
	at io.appium.java_client.AppiumDriver.startSession(AppiumDriver.java:313)
	at org.openqa.selenium.remote.RemoteWebDriver.&lt;init&gt;(RemoteWebDriver.java:173)
	at io.appium.java_client.AppiumDriver.&lt;init&gt;(AppiumDriver.java:101)
	at io.appium.java_client.AppiumDriver.&lt;init&gt;(AppiumDriver.java:113)
	at io.appium.java_client.ios.IOSDriver.&lt;init&gt;(IOSDriver.java:101)
	at com.briskpe.framework.core.DriverFactory.createIOSDriver(DriverFactory.java:120)
	at com.briskpe.framework.core.DriverFactory.createDriver(DriverFactory.java:52)
	at com.briskpe.framework.core.DriverFactory.createDriver(DriverFactory.java:40)
	at com.briskpe.tests.BaseTest.setUp(BaseTest.java:39)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:842)
Caused by: java.io.UncheckedIOException: java.net.ConnectException
	at org.openqa.selenium.remote.http.jdk.JdkHttpClient.execute0(JdkHttpClient.java:500)
	at io.appium.java_client.internal.filters.AppiumIdempotencyFilter.lambda$apply$0(AppiumIdempotencyFilter.java:35)
	at io.appium.java_client.internal.filters.AppiumUserAgentFilter.lambda$apply$0(AppiumUserAgentFilter.java:86)
	at org.openqa.selenium.remote.http.Filter.lambda$andFinally$1(Filter.java:55)
	at org.openqa.selenium.remote.http.jdk.JdkHttpClient.lambda$executeAsync$2(JdkHttpClient.java:387)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
	... 4 more
Caused by: java.net.ConnectException
	at java.net.http/jdk.internal.net.http.HttpClientImpl.send(HttpClientImpl.java:574)
	at java.net.http/jdk.internal.net.http.HttpClientFacade.send(HttpClientFacade.java:123)
	at org.openqa.selenium.remote.http.jdk.JdkHttpClient.execute0(JdkHttpClient.java:459)
	... 9 more
Caused by: java.net.ConnectException
	at java.net.http/jdk.internal.net.http.common.Utils.toConnectException(Utils.java:1055)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.connectAsync(PlainHttpConnection.java:198)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.checkRetryConnect(PlainHttpConnection.java:230)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.lambda$connectAsync$1(PlainHttpConnection.java:206)
	at java.base/java.util.concurrent.CompletableFuture.uniHandle(CompletableFuture.java:934)
	at java.base/java.util.concurrent.CompletableFuture$UniHandle.tryFire(CompletableFuture.java:911)
	at java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1773)
	... 3 more
Caused by: java.nio.channels.ClosedChannelException
	at java.base/sun.nio.ch.SocketChannelImpl.ensureOpen(SocketChannelImpl.java:195)
	at java.base/sun.nio.ch.SocketChannelImpl.beginConnect(SocketChannelImpl.java:760)
	at java.base/sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:848)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.lambda$connectAsync$0(PlainHttpConnection.java:183)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:569)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.connectAsync(PlainHttpConnection.java:185)
	... 9 more
... Removed 21 stack frames</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p>
<h3 id="m36">com.briskpe.tests.DashBoardTest#tearDown</h3><table class="result"><tr class="param"><th>Parameter #1</th></tr><tr class="param stripe"><td>[TestResult name=clickSkipButtonTest status=SKIP method=DashBoardTest.clickSkipButtonTest()[pri:1, instance:com.briskpe.tests.DashBoardTest@7e990ed7] output={null}]</td></tr><tr><th>Exception</th></tr><tr><td><div class="stacktrace">java.lang.NullPointerException: Cannot invoke &quot;com.aventstack.extentreports.ExtentTest.skip(String)&quot; because &quot;com.briskpe.tests.BaseTest.test&quot; is null
	at com.briskpe.tests.BaseTest.tearDown(BaseTest.java:64)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:842)
... Removed 22 stack frames</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p>
<h3 id="m37">com.briskpe.tests.DashBoardTest#tearDown</h3><table class="result"><tr class="param"><th>Parameter #1</th></tr><tr class="param stripe"><td>[TestResult name=verifyUserCanNavigateToSideMenuItems status=SKIP method=DashBoardTest.verifyUserCanNavigateToSideMenuItems()[pri:2, instance:com.briskpe.tests.DashBoardTest@7e990ed7] output={null}]</td></tr><tr><th>Exception</th></tr><tr><td><div class="stacktrace">java.lang.NullPointerException: Cannot invoke &quot;com.aventstack.extentreports.ExtentTest.skip(String)&quot; because &quot;com.briskpe.tests.BaseTest.test&quot; is null
	at com.briskpe.tests.BaseTest.tearDown(BaseTest.java:64)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:842)
... Removed 22 stack frames</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p>
<h3 id="m38">com.briskpe.tests.DashBoardTest#tearDownTestWithScreenshot</h3><table class="result"><tr class="param"><th>Parameter #1</th></tr><tr class="param stripe"><td>[TestResult name=clickSkipButtonTest status=SKIP method=DashBoardTest.clickSkipButtonTest()[pri:1, instance:com.briskpe.tests.DashBoardTest@7e990ed7] output={null}]</td></tr><tr><th>Exception</th></tr><tr><td><div class="stacktrace">java.lang.NullPointerException: Cannot invoke &quot;com.aventstack.extentreports.ExtentTest.skip(String)&quot; because &quot;com.briskpe.tests.DashBoardTest.test&quot; is null
	at com.briskpe.tests.DashBoardTest.tearDownTestWithScreenshot(DashBoardTest.java:89)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:842)
... Removed 22 stack frames</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p>
<h3 id="m39">com.briskpe.tests.DashBoardTest#tearDownTestWithScreenshot</h3><table class="result"><tr class="param"><th>Parameter #1</th></tr><tr class="param stripe"><td>[TestResult name=verifyUserCanNavigateToSideMenuItems status=SKIP method=DashBoardTest.verifyUserCanNavigateToSideMenuItems()[pri:2, instance:com.briskpe.tests.DashBoardTest@7e990ed7] output={null}]</td></tr><tr><th>Exception</th></tr><tr><td><div class="stacktrace">java.lang.NullPointerException: Cannot invoke &quot;com.aventstack.extentreports.ExtentTest.skip(String)&quot; because &quot;com.briskpe.tests.DashBoardTest.test&quot; is null
	at com.briskpe.tests.DashBoardTest.tearDownTestWithScreenshot(DashBoardTest.java:89)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:842)
... Removed 22 stack frames</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p>
<h3 id="m40">com.briskpe.tests.LoginTest#setUp</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th></tr><tr class="param stripe"><td>public void com.briskpe.tests.LoginTest.shouldLoginWithValidMobile()</td><td>WEB</td></tr><tr><th colspan="2">Exception</th></tr><tr><td colspan="2"><div class="stacktrace">org.openqa.selenium.SessionNotCreatedException: Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure. 
Host info: host: &apos;VISHALSHUKLA&apos;, ip: &apos;10.188.60.117&apos;
Build info: version: &apos;4.34.0&apos;, revision: &apos;707dcb4246*&apos;
System info: os.name: &apos;Windows 11&apos;, os.arch: &apos;amd64&apos;, os.version: &apos;10.0&apos;, java.version: &apos;17.0.12&apos;
Driver info: io.appium.java_client.ios.IOSDriver
Command: [null, newSession {capabilities=[Capabilities {appium:automationName: Flutter, appium:bundleId: com.briskpe.app, platformName: IOS}]}]
Capabilities {appium:automationName: Flutter, appium:bundleId: com.briskpe.app, platformName: IOS}
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:562)
	at io.appium.java_client.AppiumDriver.startSession(AppiumDriver.java:313)
	at org.openqa.selenium.remote.RemoteWebDriver.&lt;init&gt;(RemoteWebDriver.java:173)
	at io.appium.java_client.AppiumDriver.&lt;init&gt;(AppiumDriver.java:101)
	at io.appium.java_client.AppiumDriver.&lt;init&gt;(AppiumDriver.java:113)
	at io.appium.java_client.ios.IOSDriver.&lt;init&gt;(IOSDriver.java:101)
	at com.briskpe.framework.core.DriverFactory.createIOSDriver(DriverFactory.java:120)
	at com.briskpe.framework.core.DriverFactory.createDriver(DriverFactory.java:52)
	at com.briskpe.framework.core.DriverFactory.createDriver(DriverFactory.java:40)
	at com.briskpe.tests.BaseTest.setUp(BaseTest.java:39)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:842)
Caused by: java.io.UncheckedIOException: java.net.ConnectException
	at org.openqa.selenium.remote.http.jdk.JdkHttpClient.execute0(JdkHttpClient.java:500)
	at io.appium.java_client.internal.filters.AppiumIdempotencyFilter.lambda$apply$0(AppiumIdempotencyFilter.java:35)
	at io.appium.java_client.internal.filters.AppiumUserAgentFilter.lambda$apply$0(AppiumUserAgentFilter.java:86)
	at org.openqa.selenium.remote.http.Filter.lambda$andFinally$1(Filter.java:55)
	at org.openqa.selenium.remote.http.jdk.JdkHttpClient.lambda$executeAsync$2(JdkHttpClient.java:387)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
	... 4 more
Caused by: java.net.ConnectException
	at java.net.http/jdk.internal.net.http.HttpClientImpl.send(HttpClientImpl.java:574)
	at java.net.http/jdk.internal.net.http.HttpClientFacade.send(HttpClientFacade.java:123)
	at org.openqa.selenium.remote.http.jdk.JdkHttpClient.execute0(JdkHttpClient.java:459)
	... 9 more
Caused by: java.net.ConnectException
	at java.net.http/jdk.internal.net.http.common.Utils.toConnectException(Utils.java:1055)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.connectAsync(PlainHttpConnection.java:198)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.checkRetryConnect(PlainHttpConnection.java:230)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.lambda$connectAsync$1(PlainHttpConnection.java:206)
	at java.base/java.util.concurrent.CompletableFuture.uniHandle(CompletableFuture.java:934)
	at java.base/java.util.concurrent.CompletableFuture$UniHandle.tryFire(CompletableFuture.java:911)
	at java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1773)
	... 3 more
Caused by: java.nio.channels.ClosedChannelException
	at java.base/sun.nio.ch.SocketChannelImpl.ensureOpen(SocketChannelImpl.java:195)
	at java.base/sun.nio.ch.SocketChannelImpl.beginConnect(SocketChannelImpl.java:760)
	at java.base/sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:848)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.lambda$connectAsync$0(PlainHttpConnection.java:183)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:569)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.connectAsync(PlainHttpConnection.java:185)
	... 9 more
... Removed 21 stack frames</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p>
<h3 id="m41">com.briskpe.tests.LoginTest#setUp</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th></tr><tr class="param stripe"><td>public void com.briskpe.tests.LoginTest.shouldNotLoginWithInvalidMobileNumber()</td><td>IOS</td></tr><tr><th colspan="2">Exception</th></tr><tr><td colspan="2"><div class="stacktrace">org.openqa.selenium.SessionNotCreatedException: Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure. 
Host info: host: &apos;VISHALSHUKLA&apos;, ip: &apos;10.188.60.117&apos;
Build info: version: &apos;4.34.0&apos;, revision: &apos;707dcb4246*&apos;
System info: os.name: &apos;Windows 11&apos;, os.arch: &apos;amd64&apos;, os.version: &apos;10.0&apos;, java.version: &apos;17.0.12&apos;
Driver info: io.appium.java_client.ios.IOSDriver
Command: [null, newSession {capabilities=[Capabilities {appium:automationName: Flutter, appium:bundleId: com.briskpe.app, platformName: IOS}]}]
Capabilities {appium:automationName: Flutter, appium:bundleId: com.briskpe.app, platformName: IOS}
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:562)
	at io.appium.java_client.AppiumDriver.startSession(AppiumDriver.java:313)
	at org.openqa.selenium.remote.RemoteWebDriver.&lt;init&gt;(RemoteWebDriver.java:173)
	at io.appium.java_client.AppiumDriver.&lt;init&gt;(AppiumDriver.java:101)
	at io.appium.java_client.AppiumDriver.&lt;init&gt;(AppiumDriver.java:113)
	at io.appium.java_client.ios.IOSDriver.&lt;init&gt;(IOSDriver.java:101)
	at com.briskpe.framework.core.DriverFactory.createIOSDriver(DriverFactory.java:120)
	at com.briskpe.framework.core.DriverFactory.createDriver(DriverFactory.java:52)
	at com.briskpe.framework.core.DriverFactory.createDriver(DriverFactory.java:40)
	at com.briskpe.tests.BaseTest.setUp(BaseTest.java:39)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:842)
Caused by: java.io.UncheckedIOException: java.net.ConnectException
	at org.openqa.selenium.remote.http.jdk.JdkHttpClient.execute0(JdkHttpClient.java:500)
	at io.appium.java_client.internal.filters.AppiumIdempotencyFilter.lambda$apply$0(AppiumIdempotencyFilter.java:35)
	at io.appium.java_client.internal.filters.AppiumUserAgentFilter.lambda$apply$0(AppiumUserAgentFilter.java:86)
	at org.openqa.selenium.remote.http.Filter.lambda$andFinally$1(Filter.java:55)
	at org.openqa.selenium.remote.http.jdk.JdkHttpClient.lambda$executeAsync$2(JdkHttpClient.java:387)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
	... 4 more
Caused by: java.net.ConnectException
	at java.net.http/jdk.internal.net.http.HttpClientImpl.send(HttpClientImpl.java:574)
	at java.net.http/jdk.internal.net.http.HttpClientFacade.send(HttpClientFacade.java:123)
	at org.openqa.selenium.remote.http.jdk.JdkHttpClient.execute0(JdkHttpClient.java:459)
	... 9 more
Caused by: java.net.ConnectException
	at java.net.http/jdk.internal.net.http.common.Utils.toConnectException(Utils.java:1055)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.connectAsync(PlainHttpConnection.java:198)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.checkRetryConnect(PlainHttpConnection.java:230)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.lambda$connectAsync$1(PlainHttpConnection.java:206)
	at java.base/java.util.concurrent.CompletableFuture.uniHandle(CompletableFuture.java:934)
	at java.base/java.util.concurrent.CompletableFuture$UniHandle.tryFire(CompletableFuture.java:911)
	at java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1773)
	... 3 more
Caused by: java.nio.channels.ClosedChannelException
	at java.base/sun.nio.ch.SocketChannelImpl.ensureOpen(SocketChannelImpl.java:195)
	at java.base/sun.nio.ch.SocketChannelImpl.beginConnect(SocketChannelImpl.java:760)
	at java.base/sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:848)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.lambda$connectAsync$0(PlainHttpConnection.java:183)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:569)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.connectAsync(PlainHttpConnection.java:185)
	... 9 more
... Removed 21 stack frames</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p>
<h3 id="m42">com.briskpe.tests.LoginTest#tearDown</h3><table class="result"><tr class="param"><th>Parameter #1</th></tr><tr class="param stripe"><td>[TestResult name=shouldLoginWithValidMobile status=SKIP method=LoginTest.shouldLoginWithValidMobile()[pri:0, instance:com.briskpe.tests.LoginTest@1c742ed4] output={null}]</td></tr><tr><th>Exception</th></tr><tr><td><div class="stacktrace">java.lang.NullPointerException: Cannot invoke &quot;com.aventstack.extentreports.ExtentTest.skip(String)&quot; because &quot;com.briskpe.tests.BaseTest.test&quot; is null
	at com.briskpe.tests.BaseTest.tearDown(BaseTest.java:64)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:842)
... Removed 22 stack frames</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p>
<h3 id="m43">com.briskpe.tests.LoginTest#tearDown</h3><table class="result"><tr class="param"><th>Parameter #1</th></tr><tr class="param stripe"><td>[TestResult name=shouldNotLoginWithInvalidMobileNumber status=SKIP method=LoginTest.shouldNotLoginWithInvalidMobileNumber()[pri:0, instance:com.briskpe.tests.LoginTest@1c742ed4] output={null}]</td></tr><tr><th>Exception</th></tr><tr><td><div class="stacktrace">java.lang.NullPointerException: Cannot invoke &quot;com.aventstack.extentreports.ExtentTest.skip(String)&quot; because &quot;com.briskpe.tests.BaseTest.test&quot; is null
	at com.briskpe.tests.BaseTest.tearDown(BaseTest.java:64)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:842)
... Removed 22 stack frames</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p>
<h3 id="m44">com.briskpe.tests.LoginTest#tearDownLoginTest</h3><table class="result"><tr class="param"><th>Parameter #1</th></tr><tr class="param stripe"><td>[TestResult name=shouldNotLoginWithInvalidMobileNumber status=SKIP method=LoginTest.shouldNotLoginWithInvalidMobileNumber()[pri:0, instance:com.briskpe.tests.LoginTest@1c742ed4] output={null}]</td></tr><tr><th>Exception</th></tr><tr><td><div class="stacktrace">java.lang.NullPointerException: Cannot invoke &quot;com.aventstack.extentreports.ExtentTest.skip(String)&quot; because &quot;com.briskpe.tests.LoginTest.test&quot; is null
	at com.briskpe.tests.LoginTest.tearDownLoginTest(LoginTest.java:71)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:842)
... Removed 22 stack frames</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p>
<h3 id="m45">com.briskpe.tests.LoginTest#tearDownLoginTest</h3><table class="result"><tr class="param"><th>Parameter #1</th></tr><tr class="param stripe"><td>[TestResult name=shouldLoginWithValidMobile status=SKIP method=LoginTest.shouldLoginWithValidMobile()[pri:0, instance:com.briskpe.tests.LoginTest@1c742ed4] output={null}]</td></tr><tr><th>Exception</th></tr><tr><td><div class="stacktrace">java.lang.NullPointerException: Cannot invoke &quot;com.aventstack.extentreports.ExtentTest.skip(String)&quot; because &quot;com.briskpe.tests.LoginTest.test&quot; is null
	at com.briskpe.tests.LoginTest.tearDownLoginTest(LoginTest.java:71)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:842)
... Removed 22 stack frames</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p>
<h3 id="m46">com.briskpe.tests.DashBoardTest#clickSkipButtonTest</h3><table class="result"><tr><th>Exception</th></tr><tr><td><div class="stacktrace">org.openqa.selenium.SessionNotCreatedException: Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure. 
Host info: host: &apos;VISHALSHUKLA&apos;, ip: &apos;10.188.60.117&apos;
Build info: version: &apos;4.34.0&apos;, revision: &apos;707dcb4246*&apos;
System info: os.name: &apos;Windows 11&apos;, os.arch: &apos;amd64&apos;, os.version: &apos;10.0&apos;, java.version: &apos;17.0.12&apos;
Driver info: io.appium.java_client.ios.IOSDriver
Command: [null, newSession {capabilities=[Capabilities {appium:automationName: Flutter, appium:bundleId: com.briskpe.app, platformName: IOS}]}]
Capabilities {appium:automationName: Flutter, appium:bundleId: com.briskpe.app, platformName: IOS}
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:562)
	at io.appium.java_client.AppiumDriver.startSession(AppiumDriver.java:313)
	at org.openqa.selenium.remote.RemoteWebDriver.&lt;init&gt;(RemoteWebDriver.java:173)
	at io.appium.java_client.AppiumDriver.&lt;init&gt;(AppiumDriver.java:101)
	at io.appium.java_client.AppiumDriver.&lt;init&gt;(AppiumDriver.java:113)
	at io.appium.java_client.ios.IOSDriver.&lt;init&gt;(IOSDriver.java:101)
	at com.briskpe.framework.core.DriverFactory.createIOSDriver(DriverFactory.java:120)
	at com.briskpe.framework.core.DriverFactory.createDriver(DriverFactory.java:52)
	at com.briskpe.framework.core.DriverFactory.createDriver(DriverFactory.java:40)
	at com.briskpe.tests.BaseTest.setUp(BaseTest.java:39)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:842)
Caused by: java.io.UncheckedIOException: java.net.ConnectException
	at org.openqa.selenium.remote.http.jdk.JdkHttpClient.execute0(JdkHttpClient.java:500)
	at io.appium.java_client.internal.filters.AppiumIdempotencyFilter.lambda$apply$0(AppiumIdempotencyFilter.java:35)
	at io.appium.java_client.internal.filters.AppiumUserAgentFilter.lambda$apply$0(AppiumUserAgentFilter.java:86)
	at org.openqa.selenium.remote.http.Filter.lambda$andFinally$1(Filter.java:55)
	at org.openqa.selenium.remote.http.jdk.JdkHttpClient.lambda$executeAsync$2(JdkHttpClient.java:387)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
	... 4 more
Caused by: java.net.ConnectException
	at java.net.http/jdk.internal.net.http.HttpClientImpl.send(HttpClientImpl.java:574)
	at java.net.http/jdk.internal.net.http.HttpClientFacade.send(HttpClientFacade.java:123)
	at org.openqa.selenium.remote.http.jdk.JdkHttpClient.execute0(JdkHttpClient.java:459)
	... 9 more
Caused by: java.net.ConnectException
	at java.net.http/jdk.internal.net.http.common.Utils.toConnectException(Utils.java:1055)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.connectAsync(PlainHttpConnection.java:198)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.checkRetryConnect(PlainHttpConnection.java:230)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.lambda$connectAsync$1(PlainHttpConnection.java:206)
	at java.base/java.util.concurrent.CompletableFuture.uniHandle(CompletableFuture.java:934)
	at java.base/java.util.concurrent.CompletableFuture$UniHandle.tryFire(CompletableFuture.java:911)
	at java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1773)
	... 3 more
Caused by: java.nio.channels.ClosedChannelException
	at java.base/sun.nio.ch.SocketChannelImpl.ensureOpen(SocketChannelImpl.java:195)
	at java.base/sun.nio.ch.SocketChannelImpl.beginConnect(SocketChannelImpl.java:760)
	at java.base/sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:848)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.lambda$connectAsync$0(PlainHttpConnection.java:183)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:569)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.connectAsync(PlainHttpConnection.java:185)
	... 9 more
... Removed 21 stack frames</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p>
<h3 id="m47">com.briskpe.tests.DashBoardTest#verifyUserCanNavigateToSideMenuItems</h3><table class="result"><tr><th>Exception</th></tr><tr><td><div class="stacktrace">java.lang.NullPointerException: Cannot invoke &quot;com.aventstack.extentreports.ExtentTest.skip(String)&quot; because &quot;com.briskpe.tests.BaseTest.test&quot; is null
	at com.briskpe.tests.BaseTest.tearDown(BaseTest.java:64)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:842)
... Removed 22 stack frames</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p>
<h3 id="m48">com.briskpe.tests.LoginTest#shouldLoginWithValidMobile</h3><table class="result"><tr><th>Exception</th></tr><tr><td><div class="stacktrace">org.openqa.selenium.SessionNotCreatedException: Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure. 
Host info: host: &apos;VISHALSHUKLA&apos;, ip: &apos;10.188.60.117&apos;
Build info: version: &apos;4.34.0&apos;, revision: &apos;707dcb4246*&apos;
System info: os.name: &apos;Windows 11&apos;, os.arch: &apos;amd64&apos;, os.version: &apos;10.0&apos;, java.version: &apos;17.0.12&apos;
Driver info: io.appium.java_client.ios.IOSDriver
Command: [null, newSession {capabilities=[Capabilities {appium:automationName: Flutter, appium:bundleId: com.briskpe.app, platformName: IOS}]}]
Capabilities {appium:automationName: Flutter, appium:bundleId: com.briskpe.app, platformName: IOS}
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:562)
	at io.appium.java_client.AppiumDriver.startSession(AppiumDriver.java:313)
	at org.openqa.selenium.remote.RemoteWebDriver.&lt;init&gt;(RemoteWebDriver.java:173)
	at io.appium.java_client.AppiumDriver.&lt;init&gt;(AppiumDriver.java:101)
	at io.appium.java_client.AppiumDriver.&lt;init&gt;(AppiumDriver.java:113)
	at io.appium.java_client.ios.IOSDriver.&lt;init&gt;(IOSDriver.java:101)
	at com.briskpe.framework.core.DriverFactory.createIOSDriver(DriverFactory.java:120)
	at com.briskpe.framework.core.DriverFactory.createDriver(DriverFactory.java:52)
	at com.briskpe.framework.core.DriverFactory.createDriver(DriverFactory.java:40)
	at com.briskpe.tests.BaseTest.setUp(BaseTest.java:39)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:842)
Caused by: java.io.UncheckedIOException: java.net.ConnectException
	at org.openqa.selenium.remote.http.jdk.JdkHttpClient.execute0(JdkHttpClient.java:500)
	at io.appium.java_client.internal.filters.AppiumIdempotencyFilter.lambda$apply$0(AppiumIdempotencyFilter.java:35)
	at io.appium.java_client.internal.filters.AppiumUserAgentFilter.lambda$apply$0(AppiumUserAgentFilter.java:86)
	at org.openqa.selenium.remote.http.Filter.lambda$andFinally$1(Filter.java:55)
	at org.openqa.selenium.remote.http.jdk.JdkHttpClient.lambda$executeAsync$2(JdkHttpClient.java:387)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
	... 4 more
Caused by: java.net.ConnectException
	at java.net.http/jdk.internal.net.http.HttpClientImpl.send(HttpClientImpl.java:574)
	at java.net.http/jdk.internal.net.http.HttpClientFacade.send(HttpClientFacade.java:123)
	at org.openqa.selenium.remote.http.jdk.JdkHttpClient.execute0(JdkHttpClient.java:459)
	... 9 more
Caused by: java.net.ConnectException
	at java.net.http/jdk.internal.net.http.common.Utils.toConnectException(Utils.java:1055)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.connectAsync(PlainHttpConnection.java:198)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.checkRetryConnect(PlainHttpConnection.java:230)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.lambda$connectAsync$1(PlainHttpConnection.java:206)
	at java.base/java.util.concurrent.CompletableFuture.uniHandle(CompletableFuture.java:934)
	at java.base/java.util.concurrent.CompletableFuture$UniHandle.tryFire(CompletableFuture.java:911)
	at java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1773)
	... 3 more
Caused by: java.nio.channels.ClosedChannelException
	at java.base/sun.nio.ch.SocketChannelImpl.ensureOpen(SocketChannelImpl.java:195)
	at java.base/sun.nio.ch.SocketChannelImpl.beginConnect(SocketChannelImpl.java:760)
	at java.base/sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:848)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.lambda$connectAsync$0(PlainHttpConnection.java:183)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:569)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.connectAsync(PlainHttpConnection.java:185)
	... 9 more
... Removed 21 stack frames</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p>
<h3 id="m49">com.briskpe.tests.LoginTest#shouldNotLoginWithInvalidMobileNumber</h3><table class="result"><tr><th>Exception</th></tr><tr><td><div class="stacktrace">org.openqa.selenium.SessionNotCreatedException: Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure. 
Host info: host: &apos;VISHALSHUKLA&apos;, ip: &apos;10.188.60.117&apos;
Build info: version: &apos;4.34.0&apos;, revision: &apos;707dcb4246*&apos;
System info: os.name: &apos;Windows 11&apos;, os.arch: &apos;amd64&apos;, os.version: &apos;10.0&apos;, java.version: &apos;17.0.12&apos;
Driver info: io.appium.java_client.ios.IOSDriver
Command: [null, newSession {capabilities=[Capabilities {appium:automationName: Flutter, appium:bundleId: com.briskpe.app, platformName: IOS}]}]
Capabilities {appium:automationName: Flutter, appium:bundleId: com.briskpe.app, platformName: IOS}
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:562)
	at io.appium.java_client.AppiumDriver.startSession(AppiumDriver.java:313)
	at org.openqa.selenium.remote.RemoteWebDriver.&lt;init&gt;(RemoteWebDriver.java:173)
	at io.appium.java_client.AppiumDriver.&lt;init&gt;(AppiumDriver.java:101)
	at io.appium.java_client.AppiumDriver.&lt;init&gt;(AppiumDriver.java:113)
	at io.appium.java_client.ios.IOSDriver.&lt;init&gt;(IOSDriver.java:101)
	at com.briskpe.framework.core.DriverFactory.createIOSDriver(DriverFactory.java:120)
	at com.briskpe.framework.core.DriverFactory.createDriver(DriverFactory.java:52)
	at com.briskpe.framework.core.DriverFactory.createDriver(DriverFactory.java:40)
	at com.briskpe.tests.BaseTest.setUp(BaseTest.java:39)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:842)
Caused by: java.io.UncheckedIOException: java.net.ConnectException
	at org.openqa.selenium.remote.http.jdk.JdkHttpClient.execute0(JdkHttpClient.java:500)
	at io.appium.java_client.internal.filters.AppiumIdempotencyFilter.lambda$apply$0(AppiumIdempotencyFilter.java:35)
	at io.appium.java_client.internal.filters.AppiumUserAgentFilter.lambda$apply$0(AppiumUserAgentFilter.java:86)
	at org.openqa.selenium.remote.http.Filter.lambda$andFinally$1(Filter.java:55)
	at org.openqa.selenium.remote.http.jdk.JdkHttpClient.lambda$executeAsync$2(JdkHttpClient.java:387)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
	... 4 more
Caused by: java.net.ConnectException
	at java.net.http/jdk.internal.net.http.HttpClientImpl.send(HttpClientImpl.java:574)
	at java.net.http/jdk.internal.net.http.HttpClientFacade.send(HttpClientFacade.java:123)
	at org.openqa.selenium.remote.http.jdk.JdkHttpClient.execute0(JdkHttpClient.java:459)
	... 9 more
Caused by: java.net.ConnectException
	at java.net.http/jdk.internal.net.http.common.Utils.toConnectException(Utils.java:1055)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.connectAsync(PlainHttpConnection.java:198)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.checkRetryConnect(PlainHttpConnection.java:230)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.lambda$connectAsync$1(PlainHttpConnection.java:206)
	at java.base/java.util.concurrent.CompletableFuture.uniHandle(CompletableFuture.java:934)
	at java.base/java.util.concurrent.CompletableFuture$UniHandle.tryFire(CompletableFuture.java:911)
	at java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1773)
	... 3 more
Caused by: java.nio.channels.ClosedChannelException
	at java.base/sun.nio.ch.SocketChannelImpl.ensureOpen(SocketChannelImpl.java:195)
	at java.base/sun.nio.ch.SocketChannelImpl.beginConnect(SocketChannelImpl.java:760)
	at java.base/sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:848)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.lambda$connectAsync$0(PlainHttpConnection.java:183)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:569)
	at java.net.http/jdk.internal.net.http.PlainHttpConnection.connectAsync(PlainHttpConnection.java:185)
	... 9 more
... Removed 21 stack frames</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p>
</body>
</html>
